---
bio-current:
  name_en: Rebecca Doerge
  email: 
    - rwdoerge@cmu.edu # 如果有多个邮箱，请都填写上
  sex: female
  birth-date: # yyyy 到年即可
  university: Carnegie Mellon University 
  school: Department of Statistics and Data Science[http://www.stat.cmu.edu/]
  major: Statistics and Biology
  title-raw: Dean, MCS / Professor of Statistics # 主页原始字符串
  title: Professor
  interests: # 分点罗列，依次以 ‘-’ 开头
    - Mathematical and Statistical Sciences
    - Biological and Plant Sciences
    - Genomics and Epigenomics
    - statistical methodology for quantitative trait loci (QTL) mapping and transcriptomics (microarrays, next-generation sequencing, single-cell)
    - the analysis of genetic mapping and (expression) QTL experimental data
    - the analysis of the epigenome
    - the development of methodology to combine data and/or results from genomic and epigenomic investigations
  homepage: 
    - http://www.stat.cmu.edu/people/faculty/rebecca-doerge # 如果有多个主页，请都填写上
    - https://www.cmu.edu/mcs/people/dean-bio.html
  github: 
  googlescholar: https://scholar.google.com/citations?user=B4WVzSEAAAAJ&hl=en  
  aminer: https://www.aminer.cn/profile/rebecca-doerge/5605be0145ce1e595e7cc95d
  status: 在职
  last-update: 2021-9-6 # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: North Carolina State University
  school: Statistics
  email: 
  date-start: 1989
  date-end: 1993
  advisor: Bruce Weir # 格式：导师名 [邮箱/网址]
  degree: phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: University of Utah
  school: Mathematics
  date-start: 1986
  date-end: 1988
  advisor: Simon Tavaré
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: University of Utah
  school: 
  major: Mathematics
  date-start: 1982
  date-end: 1986
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: 
  research: 
  software: 
  project: 
  blog: 
  arxiv: 
  linkedin: 
  weibo:
  twitter:
  wikipedia: https://en.wikipedia.org/wiki/Rebecca_Doerge
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: 
  university: Carnegie Mellon University 
  school: Mellon College of Science[https://www.cmu.edu/mcs/index.html]
  major: Statistics and Biology
  email: rwdoerge@cmu.edu
  homepage: https://www.cmu.edu/mcs/people/dean-bio.html # 机构内学者主页
  date-start: 2016-8
  title: Glen de Vries Dean
  type: 
job-faculty-2: 
  university: Carnegie Mellon University 
  school: Dietrich College of Humanities and Social Sciences[http://www.hss.cmu.edu/]
  major: Statistics and Biology
  email: rwdoerge@cmu.edu
  homepage: http://www.stat.cmu.edu/people/faculty/rebecca-doerge # 机构内学者主页
  date-start: 
  title: Professor
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Rebecca Doerge](http://www.stat.cmu.edu/sites/default/files/faculty_pictures/doerge-pic-crop.png)

# Biography[English]
Rebecca Doerge is the Glen de Vries Dean of the Mellon College of Science at Carnegie Mellon University and a member of the Dietrich College of Humanities and Social Sciences' Department of Statistics and Data Science and the Mellon College of Science’s Department of Biological Sciences.

Doerge's research program focuses on statistical bioinformatics, an interdisciplinary component of bioinformatics that brings together many scientific disciplines for the purpose of asking, answering, and disseminating biologically interesting information in the quest to understand the ultimate function of DNA and epigenomic associations.

Doerge is an elected Fellow of the American Statistical Association (2007) and the American Association for the Advancement of Science (AAAS) (2007), and a Fellow of the Committee on Institutional Cooperation (2009). She declined the American Council on Education Fellowship in 2016 to become dean of the Mellon College of Science. She is also chair-elect of the AAAS Statistics Section (2019); a member of the Board of Trustees for the National Institute of Statistical Sciences and the Mathematical Biosciences Institute; a member of the Engineering External Review Committee at Lawrence Livermore National Laboratory; and a member of the Global Open-Source Breeding Informatics Initiative Advisory Board.

She has published two books and more than 120 scientific articles. She has mentored many students, including 24 doctoral students who have successfully defended their theses.

Doerge was born and raised in upstate New York. As a first generation student, she studied theoretical mathematics at the University of Utah. It was there that she gained interest and experience in both computing and human genetics. Doerge obtained her Ph.D. in statistics from North Carolina State University under the direction of Bruce Weir. She was a postdoctoral fellow with Gary Churchill at Cornell University, studying biometry and plant breeding.

Prior to joining Carnegie Mellon University, Doerge was the Trent and Judith Anderson Distinguished Professor of Statistics at Purdue University. She joined Purdue University in 1995 and held a joint appointment between the Colleges of Agriculture (Department of Agronomy) and Science (Department of Statistics). She was the recipient of Purdue’s Teaching for Tomorrow Award (1996), University Scholar Award (2001-06), and Provost's Award for Outstanding Graduate Faculty Mentor (2010).

# Interests[English]
  - Mathematical and Statistical Sciences
  - Biological and Plant Sciences
  - Genomics and Epigenomics
  - statistical methodology for quantitative trait loci (QTL) mapping and transcriptomics (microarrays, next-generation sequencing, single-cell)
  - the analysis of genetic mapping and (expression) QTL experimental data
  - the analysis of the epigenome
  - the development of methodology to combine data and/or results from genomic and epigenomic investigations

# Education[English]
**B.S. in Mathematics, 1982-1986**

University of Utah, Salt Lake City, Utah

**M.Stat. in Mathematics, 1986-1988**

University of Utah, Salt Lake City, Utah  
Advisor: Simon Tavaré  
Thesis: _Information of Fixed Cluster Samples for Genetic Traits Involving Mixtures of Distributions_

**Ph.D. in Statistics, 1989-1993**

North Carolina State University, Raleigh, North Carolina  
Minor in Genetics  
Advisor: Bruce Weir  
Dissertation: _Statistical Methods for Locating Quantitative Trait Loci with Molecular Markers_

**Postdoctoral Fellow, 1993-1995**

Cornell University, Ithaca, New York  
Sponsor: Gary Churchill (Department of Plant Breeding and Biometry)

# Awards[English]
- **1990-1993** NSF Fellowship in Mathematics and Molecular Biology (UC Berkeley)
- **1990-1992** Gertrude Cox Fellowship (NC State University)
- **1991** Mu Sigma Rho membership (NC State University)
- **1992** Cold Spring Harbor Laboratory award to attend Molecular Markers for Plant Breeding
- **1994** Instit. for Math. and its Appl. award to attend the Mol. Bio. program (Univ. MN)
- **1996** Teaching for Tomorrow Award (Purdue University)
- **1997** Outstanding Asst Prof. for Excellence in Teaching and Research (Science, Purdue)
- **1998** Outstanding Teacher of Undergrads, School of Science (Purdue University)
- **1998-1999** Who's Who Among America's Teachers (student nominated, Purdue University)
- **2001** Elected Member Gamma Sigma Delta
- **2002** James Madison University Visiting Scholar
- **2001-2006** University Scholar (Purdue University)
- **2007** College of Science Graduate Student Mentoring Award (Purdue University)
- **2007** Elected Member Sigma Xi
- **2007** Elected Fellow of the American Statistical Association
- **2007** Elected Fellow of the American Association for the Advancement of Science (AAAS)
- **2009** Fellow, Committee on Institutional Cooperation (CIC)
- **2010** Provost's Award for Outstanding Graduate Faculty Mentor (Purdue University)
- **2011** Trent and Judith Anderson Distinguished Professor of Statistics (Purdue University)
- **2015** College of Science Leadership Award (Purdue University)
- **2017** Glen de Vries Dean’s Chair in the Mellon College of Science (Carnegie Mellon University)

# Talks[English]

Total: Purdue University Presentations (51); Non-Purdue Presentations (186)

1. Determining Statistically Significant Changes in Gene Expression. (February 2002).

Advances in Genome Biology and Technology, Marco Island, FL (invited).

1. Determining Statistically Significant Changes in Gene Expression. (April 2002). Sloan

Kettering Cancer Center, New York, NY (invited).

1. Determining Statistically Significant Changes in Gene Expression. (May 2002). Evry

Genomics Institute and Genoscope, Evry, France (invited).

1. Determining Statistically Significant Changes in Gene Expression. (May 2002). USDA,

Stoneville, MS (invited).

1. Challenges in Bioinformatics. (June 2002). Workshop on the National Plant Genome

Initiative: 2003-2008. National Academy of Sciences, Washington, DC (invited).

1. Locating QTL. (June 2002, 3 days). The Statistical Genetics Institute, Department of

Statistics, North Carolina State University. Raleigh, NC (invited).

1. QTL Analysis Meets Microarray Analysis. (August 2002). The Joint Statistics Meetings.

New York City, NY (invited).

1. What in the World Does a Statistician Know About Genetics, Farming, and Microbes?

(August 2002). Indiana Top Farmers. Purdue University, West Lafayette, IN (invited).

1. QTL Analysis Meets Microarray Analysis. (August 2002). Bioinformatics Seminar, Purdue University, West Lafayette, IN (invited).
2. Design and Analysis of Microarray Experiments. (September 2002). National Science Foundation Plant Genome Meeting, Washington, DC (invited).
3. Old Methods for New Technology. (October 2002). Genomics Conference. Purdue University, West Lafayette, IN (invited).
4. Determining Statistically Significant Changes in Gene Expression. (October 2002). Worchester Polytech. Institute, Worchester, MA (invited).
5. Locating and Understanding Genes in Plants Using Statistics. (October 2002). James Madison University, Harrisonburg, VA (invited).
6. An Introduction to Statistical Genetics/Genomics. (October 2002). VIGRE Seminar: Explore Statistics and Mathematics Research, Purdue University, West Lafayette, IN (invited).
7. Determining Statistically Significant Changes in Gene Expression. (October 2002). Department of Statistics, University of Georgia, Athens, GA (invited).
8. Locating QTL. (December 2002, 3 days). The Statistical Genetics Institute, Trinity College, Dublin, Ireland (invited).

1. Bioinformatics: An Opportunity for Statisticians to Have an Impact on Biology. (De¬cember 2002). Department of Statistics, National University of Ireland, Galway, Ireland (invited).
2. QTL Analysis for Gene Expression Studies. (January 2003). Department of Statistics, University of New Mexico, Albuquerque, NM (invited).
3. Statistical Genetics. (February 2003). Chaired Session. Gordon Conference on Quan¬titative Genetics and Genomics. Ventura, GA (invited).
4. Old Methods for New Ideas: Expression Level Polymorphisms. (March 2003). Stadler Genetics Symposium. University of Missouri, Columbia (invited).
5. Tutorial on microarray analysis (with Bruce Craig). (June 2003). 7th Purdue Sympo¬sium on Statistics, West Lafayette, IN (invited).
6. Locating QTL. (June 2003, 3 days). The Statistical Genetics Institute, Department of Statistics, North Carolina State University. Raleigh, NG (invited).
7. Locating QTL. (July 2003, 2 days). The Southern Statistical Genetics Institute, Ge¬netics Congress, University of Melbourne, Australia (invited).
8. Functional genomics of quantitative traits: expression level polymorphism of QTL. (August 2003) Joint Statistical Meetings, San Francisco, GA (invited).
9. Using Statistics to Understand the Location and Behavior of Genes. (September 2003). VIGRE Seminar, Purdue University, West Lafayette, IN (invited).
10. The importance of experimental design and statistical analysis when analyzing microar¬ray data. (September 2003). National Science Foundation, Washington, DC (invited).
11. Old Methods for New Ideas: Expression Level Polymorphism. (September 2003). Michigan State University, Lancing, MI (invited).
12. Using expression level polymorphism (ELP) to elucidate regulatory regions in Ara- bidopsis. (November 2003). American Society for Human Genetics. Los Angles, GA (invited).
13. Functional genomics of quantitative traits: old methods for new ideas. (January 2004). The International Plant and Animal Genome Conference. San Diego, GA (invited).
14. Using functional genomics to identify candidate genes. (February 2004). Wyethe. Boston, MA (invited).
15. QTL mapping in experimental systems. (February 2004). USDA-ARS, University of California, Davis, GA (invited).
16. Functional genomics of quantitative traits. (March 2004). University of Edinburgh, Edinburgh, Scotland (invited).
17. Statistical Bioinformatics: Molecular dissection of quantitative trait loci. (March 2004). Purdue University Presidential Reviews of the School of Science, (invited).
18. QTL mapping. (June 2004). Summer Institute in Statistical Genetics, NC State

University, Raleigh, NC (invited).

1. New methods for old ideas: expression level polymorphism. (June 2004). Symposium Honoring Bruce Weir, NC State University, Raleigh, NC (invited).
2. QTL mapping. (July 2004). European Summer Institute in Statistical Genetics, Uni¬versity of Faro, Faro, Portugal (invited).
3. Locating genes using Statistics. (September 2004). VIGRE Kick-off Seminar, Purdue University, WL, IN (invited).
4. Identifying determinants of expression level polymorphism. (September 2004). CIGENE -Centre for Integrative Genetics, Agricultural University of Norway, Oslo, Norway (in¬vited).
5. Identifying determinants of expression level polymorphism. (October 2004). 12th New Phyt Symposium. Gatlinburg, TN (invited).
6. Mapping Gene Regulation: QTLs and Microarrays. (December 2004). Banbury Center. CSHL, NY (invited).
7. Meta-analysis combines Affymetrix microarray results across laboratories. (January 2005). The International Plant and Animal Genome Conference. San Diego, GA (in¬vited).
8. Microarray analysis for Polyploids. (April 2005). Banbury Center. CSHL, NY (invited).
9. Statistical Analysis of Microarray Data. What are we really testing? (June 2005). American Association for Cancer Research. Focus Group. Landsdowne Resort, VA (invited).
10. Statistical Bioinformatics. (May 2005). Dean of Science, Purdue University, Advisory Board, (invited).
11. QTL mapping. (June 2005). Summer Institute in Statistical Genetics, NC State University, Raleigh, NC (invited).
12. QTL mapping. (July 2005). Asian Summer Institute in Statistical Genetics, Yonsei University, Seoul, Korea (invited).
13. Using statistics to understand the location and behavior of genes. (October 2005). BYU -Idaho, Idaho Falls, Idaho (invited).
14. New Advances in Detecting Cancer. (November 2005). Dean of Science and Women in Science Program, Purdue University (invited).
15. Functional genomics of quantitative traits: Expression level polymorphisms of QTLs affecting disease resistance pathways in arabidopsis. (April 2006). Cold Spring Harbor Laboratory In House (invited).
16. QTL mapping, permutation thresholds, and e-QTLs. (June 2006). Noble Foundation (invited).
17. QTL mapping. (June 2006). Summer Institute in Statistical Genetics, University of

Washington, Seattle, WA (invited).

1. QTL mapping. (July 2006). European Summer Institute in Statistical Genetics, Uni¬versity of Aarhus, Aarhus, Denmark (invited).
2. Functional genomics of quantitative traits: Expression level polymorphisms of QTLs affecting disease resistance pathways in Arabidopsis. (September 2006). Department of Biostatistics, Washington University, St. Louis, MO (invited).
3. QTL mapping, permutation thresholds, and e-QTLs. (October 2006). USDA Ag. Station, Lubbock, TX (invited).
4. Functional genomics of quantitative traits: Expression level polymorphisms of QTLs affecting disease resistance pathways in Arabidopsis. (November 2006). Wellcome Trust Conference on Crop Genomics Trait Analysis and Breeding, Hinxton, UK (invited).
5. Statistical Issues in Identifying Epigenomic Changes. (January 2007). Genentech, So. San Francisco, CA (invited).
6. Functional Genomics of Quantitative Traits: Expression Level Polymorphisms of QTLs Affecting Disease Resistance Pathways in Arabidopsis (eQTL). (February 2007). De¬partment of Statistics, Indiana University, Bloomington, IN (invited).
7. Functional genomics of quantitative traits: Expression level polymorphisms of QTLs affecting disease resistance pathways in Arabidopsis. (February 2007). Gordon Confer¬ence on Quantitative Genetics and Genomics, Ventura, CA (invited).
8. Introduction to QTL mapping. (April 2007). Applied Statistics in Agriculture Keynote Speaker, Kansas State University, Manhattan, KS (invited).
9. Statistical Issues in the Analysis of Epigenetic Data. (May 2007). The International Conference on Polyploidy, Heterosis, and Epigenetics. Beijing, China (invited).
10. Functional Genomics of Quantitative Traits: Expression Level Polymorphisms of QTLs Affecting Disease Resistance Pathways in Arabidopsis (eQTL). (May 2007). Work¬shop: Addiction, Microarrays, and Gene Discovery. National Institute on Drug Abuse (NIDA). Washington, DC (invited).
11. QTL mapping. (June 2007). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
12. QTL mapping. (August 2007). Asian Summer Institute in Statistical Genetics, Na¬tional University, Seoul, Korea (invited).
13. QTL mapping. (September 2007). European Summer Institute in Statistical Genetics, University of Leige, Leige, Belgium (invited).
14. Part 1: How to Find, Apply, and Interview for a Job (Academic, Industry, etc.). (September 2007). Bioinformatics Seminar. Purdue University (Graduate Student Organization invited).
15. What Does Statistics Have to do with Genetics? (October 2007). Dean&#39;s Honor Semi¬nar, Purdue University (invited).
16. Statistical Issues in Identifying Epigenomic Changes. (November 2007). First Year Statistics Graduate Seminar, Purdue University (invited).
17. Functional Genomics of Quantitative Traits: Expression Level Polymorphisms of QTLs Affecting Disease Resistance Pathways in Arabidopsis (eQTL). (November 2007). De¬partment of Statistics, Clemson University, SC (invited).
18. Statistics in the Era of &#39;Omics. (November 2007). American Society of Agronomy, New Orleans, LA (invited).
19. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidopsis (November 2007). Dupont. Wilmington, DE (invited).
20. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidopsis (November 2007). Cornell University, Ithaca, NY (invited).
21. Part 2: How to Preparation, Interview, Negotiate, and Accept/Decline a Job (Aca¬demic, Industry, etc.). (December 2007). Bioinformatics Seminar. Purdue University (Graduate Student Organization invited).
22. Statistical Issues When Analyzing Epigenetic Data. (February 2008). 4th Course on Epigenetics. Institute Curie, Paris, France (invited).
23. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidopsis. (March 2008). South African Genetics Congress. University of Pretoria, Pretoria, South Africa (invited).
24. Interdisciplinary Research and the Importance of Networking. (April 2008). Women in Science Program. Purdue University (invited).
25. Statistical Issues When Analyzing Epigenomic Data. (April 2008). Epigenetics Work¬shop. Foundation des Treilles. Chemin des Treilles, France (invited).
26. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidopsis. (June 2008). Women in Genomics Science (WIGS). University of Washington, Seattle, WA.
27. QTL mapping. (June 2008). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
28. The Importance of Statistics in the Era of &#39;Omics. (September 2008). 9th New Phytol- ogist Symposium - Physiological Sculpture of Plants: new visions and capabilities for crop development. Timberline Lodge, Mount Hood, OR.
29. What Does Statistics Have to do with Genetics? (October 2008). Dean&#39;s Honor Semi¬nar, Purdue University (invited).
30. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidop¬sis. (November 2008). Australiasia Microarray and Associated Technology meeting. Dunedin, New Zealand.
31. QTL mapping. (December 2008). Summer Institute in Statistical Genetics, University of Auckland, Auckland, NZ (invited).
32. Whole Genome Expression Quantitative Trait Loci (eQTL) Analysis of Arabidopsis. (January 2009). Dow Agro Sciences, Indianapolis, IN (invited).
33. Statistical Issues in the Analysis of Epigenomic Data Gained From Array Technologies. (January 2009). Plant and Animal Genome Meeting. San Diego, GA (invited).
34. Introduction to QTL Mapping in Experimental Populations. (February 2009.) Bioin¬formatics Seminar. Purdue University, West Lafayette, IN (invited).
35. eQTL Mapping in Arabidopsis. (February 2009). Bioinformatics Seminar. Purdue University, West Lafayette, IN (invited).
36. The Design and Analysis of Gene Expression Experiments in Maize Polyploids. (May

1. . International Conference on Polyploidy, Hybridization and Biodiversity. Palais du Grand Large, Saint Malo, France (invited).

1. QTL mapping. (June 2009). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
2. Statistical Challenges in Modelling Expression Quantitative Traits and Gene Networks. (July 2009). 24th International Workshop on Statistical Modelling. Cornell University, Ithaca, NY (invited).
3. Developing Statistics Programs for Majors Outside of Statistics and Expanding Inter¬disciplinary Cooperation. (August 2009) Heads/Chairs Workshop, American Statistical Association. Arlington, VA (invited).
4. Statistical Issues and Challenges of Epigenomics. (August 2009). Impact of High¬Dimensional Data on Molecular Epidemiology and Statistical Genomics Invited Session. Joint Statistics Meetings, Washington, DC (invited).
5. QTL mapping. (September 2009). Summer Institute in Statistical Genetics, University of Liege. Liege, Belgium (invited).
6. Gene networks from eQTL; Are We There Yet? (September 2009). Genes, Systems and Phenotypes, Modelling Networks from Sequence to Consequence GENESYS meeting at ECCS09 (keynote speaker). University of Warwick, Warwick, UK.
7. Statistics, Epigenomics, and Cancer. (September 2009). Exploring Statistical Sciences Research Seminar. Purdue University (invited).
8. What is a Statistician and What Do They Do? And What does it have to do with genetics? (November 2009). Freshman Mathematics Seminar. Purdue University (in¬vited).
9. What is the Difference Between CVs and Resumes? (January 2010). Women In Science Program (WISP), Purdue University (invited).
10. Part 1: Beginning a Job Search: Academic, Industry, Government... (January 2010).

Bioinformatics Seminar, Purdue University (invited).

1. Part 2: The Job Interview and What to do When You Get an Offer (Academic, Industry, etc.). (March 2010). Bioinformatics Seminar, Purdue University (invited).
2. QTL mapping. (June 2010). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
3. Modeling Differential Expression in RNA-Seq Data. (August 2010). Impact of Statis¬tics on Next Generation Sequencing, Joint Statistics Meetings, Vancouver, BC (invited).
4. QTL mapping. (August 2010). Summer Institute in Statistical Genetics, Peking University, Beijing, China (invited).
5. Statistical Issues When Modeling RNA-Seq Data for Differential Expression. (October

1. . Department of Biostatistics, Indiana University Purdue University, Indianapolis (IUPUI), IN (invited).

1. Statistical Issues When Modeling RNA-Seq Data for Differential Expression. (Novem¬ber 2010). Department of Applied and Computational Mathematics and Statistics, Notre Dame University, South Bend, IN (invited).
2. Statistical Issues When Modeling RNA-Seq Data for Differential Expression. (Novem¬ber 2010). Cold Spring Harbor Laboratory. Cold Spring Harbor, NY (invited).
3. Statistical Issues When Modeling RNA-Seq Data for Differential Expression. (January

1. . Next Generation Sequencing Symposium. Indianapolis, IN (invited).

1. Statistical Issues When Modeling RNA-Seq Data for Differential Expression. (March

1. . International Biometrics Society Eastern North American Region meeting. Mi¬ami, FL (invited).

1. High-dimensional QTL mapping in Arabidopsis. (March 2011). Kavli Royal Society International Centre, Buckinghamshire, UK (invited).
2. A Two-Stage Poisson Model for Testing RNA-Seq Data. (April 2011). Departments of Plant Science and Statistics; University of Missouri, Columbia, MO (invited).
3. A Two-St age Poisson Model for Testing RNA-Seq Data. (April 2011). Department of Biological Statistics and Computational Biology. Cornell University, Ithaca, NY (invited).
4. QTL mapping. (June 2011). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
5. Part 1: Beginning a Job Search: Academic, Industry, Government... (August 2011). Bioinformatics Seminar, Purdue University, West Lafayette, IN (invited).
6. Part 2: The Job Interview and What to do When You Get an Offer (Academic, Industry, etc.). (September 2011). Bioinformatics Seminar, Purdue University, West Lafayette, IN (invited).
7. QTL mapping. (September 2011). European Summer Institute in Statistical Genetics,

University of Washington, Seattle, WA (invited).

1. Large Scale Epigenome Analysis. (October 2011). International Breast Cancer and Nutrition Conference. Rennes, France (invited).
2. Statistics, Technology, Epigenomics and our Future. (October 2011). Department of Mathematics, Trinity University, San Antonio, TX (invited).
3. Statistical Issues and the Analysis of Next-Generation Sequencing Data. (January

1. . Plant and Animal Genome XX, San Diego, CA (invited).

1. Modeling Next-Generation Sequencing Data and Related Statistical Issues. (March

1. . Department of Statistics. Iowa State University. Ames, IA (invited).

1. Modeling N ext- Generat ion Sequencing Data and Related Statistical Issues. (April

1. . Department of Statistics. University of Nebraska, Lincoln, NB (invited).

1. A Conversation About the Role(s) of Statistical Bioinformatics in a Department of Statistics, and at a University. (April 2012). Department of Statistics. University of Nebraska, Lincoln, NB (invited).
2. Job Interviewing Skills. (April 2012). Purdue University Graduate Student Associa¬tion. Purdue University. West Lafayette, IN (invited).
3. Modeling Next-Generation Sequencing Data and Related Statistical Issues. (May

1. . Department of Statistics. Northwestern University, Evanston, IL (invited).

1. QTL mapping. (June 2012). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
2. Modeling Next-Generation Sequencing Data and Related Statistical Issues. (August 2012). Department of Statistics and the Department of Genetics. University of Georgia, Athens, GA (invited).
3. Self Promotion Techniques and Interview Skills. (September 2012). Women in Science Program. Purdue University. West Lafayette, IN (invited).
4. Statistical Issues in Plant Genomic Data. (January 2013). SAS Plant Genome Work¬shop. Plant and Animal Genome Conference. San Diego, CA (invited).
5. Part 1: Beginning a Job Search: Academic, Industry, Government... (January 2013). Bioinformatics Seminar. Purdue University.
6. Part 2: The Job interview and What to do When You Get an Offer (academic, industry, etc.). (January 2013). Bioinformatics Seminar. Purdue University.
7. Introduction to Systems Genetics Session. (February 2013). Gordon Conference on Quantitative Genetics and Genomics. Galveston Island, TX (invited).
8. Statistical Bioinformatics: Statistical Analysis of &#39;Omic Data. (March 2013). Qatar Computing Research Institute. Doha, Qatar (invited).
9. The Job Interview and What to do When You Get an Offer (academic, industry, etc.). (March 2013). Postdoctoral Association, Purdue University (invited).
10. Conversation about Academics. (March 2013). Association for Women in Mathemat¬ics. Purdue University.
11. Modeling N ext- Generat ion Sequencing Data and Related Statistical Issues. (April

1. . Department of Mathematics. Ball State University, Munice, IN (invited).

1. QTL mapping. (July 2013). Summer Institute in Statistical Genetics, University of Washington, Seattle, WA (invited).
2. Analyzing N ext- Generat ion Sequencing Data and Related Statistical Issues. (August

1. . Department of Genetics and Department of Statistics. University of Georgia, Athens, GA (invited).

1. Common Themes in Statistical Bioinformatics Analysis. (August 2013). Joint Statis¬tics Meetings. Montreal, Canada (invited).
2. Analyzing Next-Generation Sequencing Data and Related Statistical Issues. (Novem¬ber 2013). Synbreed. Munich, Germany (invited).
3. Correcting for Amplification Bias in N ext- Generat ion Sequencing Data. (November

1. . &quot;Impact of Large scale Omic Data on Statistical and Quantitative Genetics&quot;, University of Washington, Seattle, WA (invited).

1. Correcting for Amplification Bias in Next-Generation Sequencing Data. (December

1. . The Ninth ICSA International Conference: Challenges of Statistical Methods for Interdisciplinary Research and Big Data, Hong Kong Baptist University, Hong Kong (invited).

1. Correcting for Amplification Bias in Next-Generation Sequencing Data. (January

1. . Plant and Animal Genome Meeting. San Diego, GA (invited).

1. Correcting for Amplification Bias in Next-Generation Sequencing Data. (February

1. . Bioinformatics Seminar Series. Purdue University (invited).

1. Challenges in Bioinformatics. (April 2014). Indiana Clinical and Translational Science Institute Retreat. Purdue University (invited).
2. MAGI: Methylation Analysis using Genome Information. (April 2014). Applied Statis¬tics in Agriculture Conference. Kansas State University. Manhattan, KS. (invited).
3. Growing your Research Program: When, Why, and How. (May 2014). Celebrating Women in Statistics: Know Your Power. Durham, NG. (invited).
4. Correcting for Amplification Bias in N ext- Generat ion Sequencing Data. (June 2014). Chengdu University. Chengdu, China, (invited).
5. QTL mapping. (July 2014). Summer Institute in Statistical Genetics, University of Washington. Seattle, WA (invited).
6. Correcting for Amplification Bias in N ext- Generat ion Sequencing Data. (August

1. . Joint Statistics Meetings. Boston, MA. (invited).

1. Analyzing N ext- Generat ion Sequencing Data and Related Statistical Issues. (August

1. . Molecular Biology Institute Summer Undergraduate Research Program. The Ohio State University. Columbus, OH.

1. Wanted: Large Complex Omic Data Seeking Quantitative Reasoning and Analysis. (September 2014). Opening Workshop on Beyond Bioinformatics. Statistics and Ap¬plied Mathematics Institute (SAMSI). Durham, NC. (invited).
2. The Impact of Academic Mentoring on the Work Environment. Provost ASSIST Program. North Carolina State University. Raleigh, NC. (invited).
3. Correcting for Amplification Bias in Next-Generation Sequencing Data. (Septem¬ber 2014). Department of Genetics, North Carolina State University. Raleigh, NC. (invited).
4. RASTA: Robust Adjustment of Sequence Tag Abundance in N ext- Generat ion Se¬quencing Data. (October 2014). Department of Plant Sciences. University of Missouri. Columbia, MO. (invited).
5. The Path to Success Isnt Always Linear: One Statisticians Not Boring and Nonlinear Story Forward. (November 2014). Department of Statistics, Undergraduate Statistics Club; North Carolina State University. Raleigh, NC. (invited).
6. The Role and Importance of Modern Interdisciplinary Statistics in Academics. (De¬cember 2014). Northeastern University. Boston, MA. (invited).
7. Part 1: Beginning a Job Search: Academic, Industry, Government... (January 2015). Purdue University (invited).
8. Wanted: Large Complex Omic Data Seeking Quantitative Reasoning and Analysis. (March 2015). The Challenge of Inference from Genome to Phenome. The Common¬wealth Scientific and Industrial Research Organization (CSIRO). Brisbane, Australia (invited).
9. Introduction to Quantitative Trait Locus (QTL) in Experimental Populations. (March

1. . Bioinformatics Seminar. Purdue University (invited).


# Work experience[English]
- **8/1990-12/1992** - NSF Research Fellow  
    North Carolina State University, Statistics Department
- **12/1992-5/1993** \- Research Assistant  
    North Carolina State University, Statistics Department
- **5/1993-7/1993** - Postdoctoral Associate  
    North Carolina State University, Statistics Department: Dr. Bruce Weir
- **8/1993-7/1995** - Instructor/Postdoctoral Associate  
    Cornell University, Biometrics Unit: Dr. Gary Churchill
- **8/1995-8/2000** - Assistant Professor of Statistics (75%) and Agronomy (25%)  
    Purdue University
- **8/2000-8/2003** \- Associate Professor of Statistics and Agronomy  
    Purdue University
- **8/2003-8/2016** \- Professor of Statistics and Agronomy  
    Purdue University
- **8/2008-7/2009** - Interim Head of Statistics  
    Purdue University, Department of Statistics
- **7/2009-6/2010** \- Associate Head of Statistics  
    Purdue University, Department of Statistics
- **7/2010-6/2015** \- Department Head of Statistics  
    Purdue University, Department of Statistics
- **10/2011-8/2016** \- _Trent and Judith Anderson Distinguished Professor of Statistics_
- **8/2016-Present** \- Dean of Mellon College of Science  
    Carnegie Mellon University
- **12/2017-Present** \- _Glen de Vries Dean's Chair in the Mellon College of Science_

# Publication[English]
Books (2); Reviewed Book Chapters (7); Reviewed Proceedings (29); Journal Publications (92) Current and former Purdue graduate students denoted by \*

1. G.A. Churchill and R.W. Doerge. 1994. Empirical threshold values for quantitative trait

mapping. Genetics. 138:963-971.

1. R.W. Doerge. 1995. Testing for linkage: phase known/phase unknown. Journal of Heredity. 86(l):61-62.
2. R.W. Doerge. 1995. The relationship between the LOD score and the analysis of variance

F-statistic when detecting QTL using single markers. Appendix 1: Locating genes associated with root morphology and drought avoidance in rice via linkage to molecular markers. M.C. Champoux, G. Wang, S. Sarkarung, D.J. Mackill, J.C. O&#39;Toole, N. Huang, S.R. McCouch. Theoretical and Applied Genetics. 90:969-981.

1. S.R. McCouch and R.W. Doerge. 1995. QTL mapping in rice. Trends in Genetics.

ll(12):482-487.

1. R.W. Doerge and G.A. Churchill. 1996. Permutation tests for multiple loci affecting a

quantitative character. Genetics. 142:285-294.

1. R.W. Doerge and A. Reba&#39;i. 1996. Significance thresholds for QTL interval mapping

tests. Heredity. 76:459-464.

1. M.A. Mutschler, R.W. Doerge, J.P. Kuai, S.C. Liu, B. Liedl, and Y. Shapiro. 1996.

QTL analysis of the production of acylsugars responsible for pest resistance in the wild tomato _lycopersicon pennellii._ Theoretical and Applied Genetics. 92:709-719.

1. R.W. Doerge. 1996. Constructing genetic maps by rapid chain delineation. Journal of

Quantitative Trait Loci. Volume 2. Article 6. http://probe.nalusda.gov:8000/otherdocs/jqtl

1. R.W. Doerge, Z-B. Zeng, and B.S. Weir. 1997. Statistical issues in the search for genes

affecting quantitative traits in experimental populations. Statistical Science. 12(3):195- 219.

1. C. Teuscher, D.M. Rhein, K.D. Livingstone, R.A. Paynter, R.W. Doerge, S.M. Nichol¬son, and R.W. Mel void. 1997. Evidence that Tmevd2 and eae3 may represent either a common locus or gene complex controlling susceptibility to immunologically mediated demyelination in micel. Journal of Immunology. 159:4930-4934.
2. G.A. Churchill and R.W. Doerge. 1997. Mapping Quantitative Trait Loci in Experi¬mental Populations. In: _Molecular Dissection of Complex Traits._ A.H. Paterson (ed). CRC Press: New York.
3. R.J. Roper, R.W. Doerge, S.B. Call, K.S.K. Yung, W.F. Hickey, and C. Teuscher. 1998. Autoimmune orchitis, epididymitis, and vastis are immunogenetically distinct lesions. The American Journal of Pathology. 152(5):1337-1345.
4. R.J. Roper, J.S. Griffith, C.R. Lyttle, R.W. Doerge, A.W. McNabb, R.E. Broadbent, and C. Teuscher. 1998. Interacting quantitative trait loci control phenotypic variation in murine estradiol-regulated responses. Endocrinology. 2(140):556-561.
5. R.W. Doerge. 1998. Statistical threshold values for locating quantitative trait loci. Pro¬ceedings of the Tenth Annual Conference on Applied Statistics in Agriculture, Kansas State University. Edited by George Milliken. 10:84-95.
6. J. Weis, B. McCracken, Y. Ma, D. Fairbairn, R. Roper, J. Weis, J. Zachery, R.W. Doerge, and C. Teuscher. 1999. Identification of quantitative trait loci governing arthritis severity and humoral responses in the murine model of Lyme disease. Journal of Immunology. 162:948-956.
7. R.J. Butterfield, E.P. Blankenhorn, R.J. Roper, J.F. Zachary, R.W. Doerge, J. Sud- weeks, J. Rose, and C. Teuscher. 1999. Genetic analysis of disease subtype and sex¬ual dimorphism in mouse EAE: Relapsing-remitting and monophasic remitting/non- relapsing EAE are immunogenetically distinct. Journal of Immunology. 162(5):3096- 3102.
8. S.A. Fennimore, G.E. Shaner, W.E. Nyquist, R.W. Doerge, and M.E. Foley. 1999. A genetic model and molecular markers for wild oat _(Avena, fatua_ L.) seed dormancy. Theoretical and Applied Genetics. (3-4):711-718.
9. C. Teuscher, R.J. Butterfield, R. Ma, J. Zachary, R.W. Doerge, and E.P. Blankenhorn.

1. Sequence polymorphisms in the chemokines _scyal_ (TCA-3), _scya2_ (Monocyte chemattractant protein (MCP-1)), and _scyal2_ (MCP-5) are candidate genes for _eae_ 7, a locus controlling the susceptibility to monophasic remitting/nonrelapsing experimental allergic encephalomyelitis. Journal of Immunology. 163:2262-2266.

1. D. Nettleton and R.W. Doerge. 2000. Accounting for variability in the use of permu¬tation testing to detect quantitative trait loci. Biometrics. 56:285-291.
2. R. Bernardo, J. Romero-Severson, J. Ziegle, J. Hauser, L. Joe, G. Hookstra, and R.W. Doerge. 2000. Parental contribution and coefficient of coancestry among maize inbreds: Pedigree, RFLP, and SSR data. Theoretical and Applied Genetics. 100:552-556.
3. E.P. Blankenhorn, R.J. Butterfield, R. Rigby, L. Cort, D. Giambrone, P. McDermott, K. McEntee, N. Solowski, N.D. Meeker, J.F. Zachary, R.W. Doerge, and C. Teuscher.

1. Genetic analysis of the influence of pertussis toxin in EAE susceptibility: an environmental agent can override genetic checkpoints. The Journal of Immunology. 164(6):3420-3425.

1. R.W. Doerge and Bruce A. Craig. 2000. Model selection for quantitative trait locus analysis in polyploids. Proceedings of the National Academy of Science. 97(14):7951- 7956.
2. R.J. Butterfield, E.P. Blankenhorn, R.J. Roper, J.F. Zachary, R.W. Doerge, and C. Teuscher. 2000. Genetic analysis of the histopathology of murine Experimental Allergic Encephaolmyelitis (EAE): identification of brain, spinal cord, and gender-specific loci controlling lesion formation and severity. American Journal of Pathology. 157(2):637- 646.
3. R.W. Doerge and Bruce A. Craig. 2000. An introduction to model selection for quantitative trait locus analysis in polyploids. Proceedings of Applied Statistics in Agriculture, Kansas State University. Edited by George Milliken, ppl3-27.
4. L.M. McIntyre, C. Coffman, and R.W. Doerge. 2001. Detection and location of a single binary trait locus in experimental populations. Genetical Research. 78:79-92.
5. R.J. Roper, J.J. Weis, B.A. McCracken, C.B. Green, Y. Ma, K.S. Weber, D. Fairbairn, R.J. Butterfield, M.R. Potter, J.F. Zachary, R.W. Doerge, and C. Teuscher. 2001. Susceptibility to experimental lyme arthritis is linked to multiple loci on chromosome 5 in four independent mouse crosses. Genes and Immunity. 2(7):388-397.
6. R.W. Doerge. 2001. Quantitative Trait Loci. Plant Sciences for Students. Macmillan Reference USA, pp!95-196.
7. J. Wilbur\*, J.K. Ghosh, C.H. Nakatsu, S.M. Brouder, and R.W. Doerge. 2001. Sta-tistical issues in the analysis of microbial communities in soil. Proceedings of Applied Statistics in Agriculture, Kansas State University. Edited by George Milliken, pp!29- 143.
8. M.A. Black\* and R.W. Doerge. 2001. Calculation of the minimum number of replicate spots required for detection of significant gene expression fold changes for cDNA mi¬croarrays. Proceedings of Applied Statistics in Agriculture, Kansas State University. Edited by George Milliken, pp!44-158.
9. B.A. Craig, O. Vitek\*, M.A. Black\*, M. Tanurdzic\*, and R.W. Doerge. 2001. Designing microarray experiments: chips, dips, flips, and skips. Proceedings of Applied Statistics in Agriculture, Kansas State University. Edited by George Milliken, pp!59-182.
10. R.W. Doerge. 2002. Mapping and analysis of quantitative trait loci in experimental populations. Nature Reviews Genetics. 3:43-52.
11. J. Wilbur\*, J.K. Ghosh, C.H. Nakatsu, S.M. Brouder, and R.W. Doerge. 2002. Variable selection in high-dimensional multivariate binary data with application to the analysis of microbial community DNA fingerprints. Biometrics. 58:378-386.
12. R.J. Roper, R.Z. Ma, J.E. Biggins, R.J. Butterfield, S.D. Michael, K.S.K. Tung, R.W. Doerge, and C. Teuscher. 2002. Interacting quantitative trait loci control loss of pe¬ripheral tolerance and susceptibility to autoimmune ovarian dysgenesis after day three thymectomy in mice. Journal of Immunology. 169:1640-1646.
13. M. Black\* and R.W. Doerge. 2002. Calculation of the minimum number of replicate spots required for detection of significant gene expression fold change in microarray experiments. Bioinformatics. 18(12):1609-1616.
14. R.J. Butterfield, R.J. Roper, D.M. Rhein, R.W. Mel void, L.Haynes, R.Z. Ma, R.W. Do¬erge, and C. Teuscher. 2002. Sex specific QTL govern susceptibility to Theiler&#39;s murine encephalomyelitis virus-induced demyelination (TMEVD). Genetics. 163(3):1041-1046.
15. B.A. Craig, M.A. Black\*, and R.W. Doerge. 2003. Gene Expression: The technology and statistical analysis. J ABES. 8(l):l-28.
16. T.C. Osborn, J.C. Pires, J.A. Birchler, Z.J. Chen, L. Comai, R.W. Doerge, and R.A. Martienssen. 2003. Understanding the causes of novel variation in polyploids. Trends in Genetics. 19(3):141-147.
17. Veena, H. Jiang\*, R.W. Doerge, and S.B. Gelvin. 2003. Differential expression of plant genes during the initial stages of Agrobacterium-mediated transformation. The Plant Journal. 35:219-236.
18. C. Coffman, R.W. Doerge, M. Wayne, and L.M. McIntyre. 2003. Intersection tests for Single marker QTL analysis can be more powerful than two marker QTL analysis. BioMed Central (BMC) Genetics. 4:10.
19. O. Abiola, J.M. Angel, P. Avner, A.A. Bachmanov, B. Bennett, E.P. Blankenhorn, D.A. Blizard, V. Bolivar, G.A. Brockmann, W.L. Casley, M. Cook, K.J. Buck, J.-F. Bureau, E.J. Chester, J.M. Cheverud, G.A. Churchill, J.C. Crabbe, W.E. Crusio, A. Darvasi, P. Demant, R.W. Doerge, R.W. Elliott, C.R. Farber, J. Flint, H. Gershenfeld, J.P. Gibson, W. Gu, G. de Haan, H. Himmelbauer, R. Hitzemann, K. Hunter, H.-C. Hsu, F. Iraqi, R.C. Jansen, T.E. Johnson, G. Kempermann, F. Lammert, L. Lu, K.F. Manly, D.B. Matthews, J.F. Medrano, M. Mehrabian, G. Mittleman, B.A. Mock, J.S. Mogil, X. Montagutelli, G. Morahan, J.D. Mountz, H. Nagase, R.S. Nowakowski, B.F. O&#39;Hara, A.V. Osadchuk, B. Paigen, A.A. Palmer, J.L. Peirce, D. Pomp, G.D. Rosen, M. Rosemann, L.C. Schalkwyk, Z. Seltzer, S. Settle, K. Shimomura, S. Shou, J.M. Sikela, L.D. Siracusa, J.L. Spearow, C. Teuscher, D.W. Threadgill, L.A. Toth, A.A. Toye, C. Vadasz, E. Wakeland, R.W. Williams, G. Van Zant, H.-G. Zhang, F. Zou, and L. Flaherty. 2003. The nature and identification of quantitative trait loci: a community&#39;s view. Nature Genetics Reviews. 4:911-916.
20. D. Cao\*, B.A. Craig, and R.W. Doerge. 2003. Interval mapping for autopolyploids. Proceedings of Applied Statistics in Agriculture, Kansas State University. Edited by

George Milliken, pp60-72.

1. D. Cao\*, T.C. Osborn, and R.W. Doerge. 2004. Correct estimation of preferential chromosome pairing in polyploids. Genome Research. 14:459-462.
2. M. Bogdan, J.K. Ghosh, and R.W. Doerge. 2004. Modifying the Schwarz Bayesian Information Criterion to locate multiple interacting quantitative trait loci. Genetics. 167:989-999.
3. H.-S. Lee, J. Wang, L. Tan, H. Jiang, M.A. Black\*, A. Madlung, L. Lukens, J.C. Pires, L. Comai, T.C. Osborn, R.W. Doerge, and Z.J. Chen. 2004. Sensitivity of 70-mer oligonucleotides and cDNAs for microarray analysis of gene expression in Arabidopsis and its related species. Plant Biotechnology Journal. 2:45-57.
4. Z.J. Chen, J. Wang, L. Tian, H.-S. Lee, J.J. Wang, M. Chen, J.J. Lee, C. Josefsson, A. Madlung, B. Watson, J.C. Pires, Z. Lippman, M. Vaughn, V. Colot, J.A. Birchler, R.W. Doerge, R.A. Martienssen, L. Comai, and T.C. Osborn. 2004. The development of an Arabidopsis model system for genome-wide analysis of polyploidy effects. Biological Journal of the Linnean Society. 82:689-700.
5. Z. Lippman, A.-V. Gendrel, M. Black\*, M. Vaughn, N. Dedhia, W.R. McCombie, K. Lavine, V. Mittal, B. May, K. Kasschau, J.C. Carrington, R.W. Doerge, V. Colot, and R. Martienssen 2004. Transposable elements mediate heterochromatin and epigenetic control. Nature. 430:471-476.
6. K. Bradley, M.B. Boulware, H. Jiang\*, R.W. Doerge, and R.L. Meisel. 2004. Sexual experience generates distinct patterns of gene expression in the nucleus accumbens and dorsal striatum of female Syrian hamsters. Genes, Brain, and Behavior. 4(l):31-44.
7. K. Kim\*, M. West, R.W. Michelmore, D. St. Clair, and R.W. Doerge. 2004. Genetic mapping of gene expression levels: expression level polymorphism analysis for dissecting regulatory networks of plant disease resistance. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp97-lll.
8. D. Cao\*, B.A. Craig, and R.W. Doerge. 2004. Bayesian quantitative trait mapping of autopolyploids. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS.
9. H. Jiang\*, J. Wang, L. Tian, Z.J. Chen, and R.W. Doerge. 2004. Statistical analysis of 70-mer oligonucleotide microarray from polyploid experiments using repeated dye¬swaps. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS.
10. The Complex Traits Consortium (see publication 40 for list of members). 2004. The collaborative cross: A community resource for the genetic analysis of complex traits. Nature Genetics. 36(ll):1133-7.
11. J. Wang, J.J. Lee, H.-S. Lee, L. Tian, M. Chen, S. Rao, E.N. Wei, R.W. Doerge, L. Comai, and Z.J. Chen. 2005. Methods for genome-wide analysis of gene expression changes in polyploids. Methods in Enzymology. 395:570-96.
12. L. Tian, M.P. Fong, J.J. Wang, N.E. Wei, H. Jiang\*, R.W. Doerge, and Z.J. Chen. 2005. Reversible histone acetylation and deacetylation mediate genome-wide, promoter¬dependent, and locus-specific changes in gene expression during plant development. Genetics. 169:337-345.
13. A. Madlung, A.P. Tyagi, B. Watson, H. Jiang\*, T. Kagochi, R.W. Doerge, R. Mar- tienssen, and L. Comai. 2005. Genomic changes in synthetic Arabidopsis polyploids. Plant Journal. 41(2):221-230.
14. D. Cao\*, B. Craig, and R.W. Doerge. 2005. A model selection based interval mapping method for autopolyploids. Genetics. 169:2371-2382.
15. M. Bogdan and R.W. Doerge. 2005. Biased estimators of genetic effects and heritability in interval mapping. Heredity. 95:476-484.
16. C. Coffman, R.W. Doerge, K.L. Simonsen, K. Nichols, C. Duarte, R. Wolfinger, and L.M. McIntyre. 2005. Model selection in binary trait locus mapping. Genetics. 170: 1281-1297.
17. J. Stevens\* and R.W. Doerge. 2005. Combining Affymetrix microarray results. BMC Bioinformatics. 6:57.
18. J. Stevens\* and R.W. Doerge. 2005. Meta-Analysis combines Affymetrix microarray results across laboratories. Comparative and Functional Genomics. 6:116-122.
19. R.A. Martienssen, R.W. Doerge, and V. Colot. 2005. Epigenomic mapping in Ara¬bidopsis using tiling microarrays. Chromosome Research. 13 3:299-308.
20. B. Munneke\*, K.A. Schlauch, K. Simonsen, W. Beavis, and R.W. Doerge. 2005. Adding confidence to gene expression clustering. Genetics. 170:2003-2011.
21. J. Stevens\* and R.W. Doerge. 2005. A Bayesian and covariate approach to combine re¬sults from multiple microarray studies. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp!33-147.
22. K. Kim\*, M. West, R.W. Michelmore, D. St. Clair, and R.W. Doerge. 2005. Old methods for new ideas: using gene expression levels to molecularly dissect complex traits. The Proceedings of the Stadler Genetics Symposium. March 2003. Edited by Perry Gustafson, Randy Shoemaker, and John Snape. Springer, NY. pp89-106.
23. J. Wang, L. Tian, H-S Lee, N.E. Wei, H. Jiang\*, B. Watson, A. Madlung, T.C. Os-born, R.W. Doerge, L. Comai, and Z.J. Chen. 2006. Genome-wide non-additive gene regulation in Arabidopsis allotetraploids. Genetics. 172:507-517.
24. D.J. Kliebenstein, M.A.L. West, H. van Leeuwen, K. Kim\*, R.W. Doerge, R.W. Michel- more, and D.A. St. Clair. 2006. Genomic survey of gene expression diversity in Ara¬bidopsis thaliana. Genetics. 172:1179-1189.
25. C. Teuscher, R.W. Doerge, P.D. Fillmore, and E.P. Blankenhorn. 2006. eae36, A locus on mouse chromosome 4 controls susceptibiltiy to experimental allergic encephalomyeli¬tis in older mice and mice immunized in the winter. Genetics. 172:1147-1153.
26. M.A.L. West, H. van Leeuwen, A. Kozik, D.J. Kliebenstein, R.W. Doerge, D.A. St. Clair, and R.W. Michelmore. 2006. High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis. Genome Research. 16:787-795.
27. D.J. Kliebenstein, M.A.L. West, H. van Leeuwen, O. Loudet, R.W. Doerge, and D.A. St. Clair. 2006. Identification of QTLs controlling gene expression networks defined a priori. BMC Bioinformatics. 7(l):308.
28. L. An\*, N. Riddle, J. Birchler, and R.W. Doerge. 2006. Clustering a series of replicated polyploid gene expression experiments. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp52-66.
29. H. Jiang\* and R.W. Doerge. 2006. A two-step multiple comparison procedure for a large number of tests and multiple treatments. Statistical Applications in Genetics and Molecular Biology. 5: 1, Article 28.
30. J. M. Ordway, J.A. Bedell, R.W. Citek, A. Nunberg, A. Garrido, R. Kendall\*, J.R. Stevens\*, D. Cao\*, R.W. Doerge, Y. Korshunova, H. Holemon, J.D. McPherson, N. Lakey, J. Leon, R.A. Martienssen, and J.A. Jeddeloh. 2006. Comprehensive DNA methylation profiling in a human cancer genome identifies novel epigenetic targets. Carcinogenesis. 27(12):2409-2423.
31. M.A.L. West, K. Kim\*, D.J. Kliebenstein, H. van Leeuwen, R.W. Michelmore, R.W. Doerge, and D.A. St. Clair. 2007. Global eQTL mapping reveals the complex genetic architecture of transcript level variation in Arabidopsis. Genetics. 175:1441-1450.
32. M. Vaughn, M. Tanurdzic, Z. Lippman, H. Jiang, R. Carrasquillo, V. Colot, R.W. Do¬erge and R.A. Martienssen. 2007. Epigenetic natural variation in Arabidopsis thaliana. Public Library of Science (PLoS) Biology. 5(7): e!74.
33. H. van Leeuwen, D.J. Kliebenstein, M.A.L. West, K. Kim\*, R. van Poecke, F. Katagiri, R.W. Michelmore, R.W. Doerge, and D.A. St. Clair. 2007. Natural variation among Arabidopsis thaliana accessions for transcriptome response to exogenous salicylic acid. The Plant Cell. 19(7):2099-2110.
34. J. Zhao, J. Wang, L. An\*, R.W. Doerge, Z.J. Chen, C. Grau, J. Meng, and T. Osborn. 2007. Analysis of gene expression profiles in response to Sclerotinia sclerotiorum in Brassica napus. Planta. 227(l):13-24.
35. J.M. Ordway, M.A. Budiman, Y. Korshunova, R.K. Maloney, J.A. Bedell, R.W. Citek, B. Bacher, S. Peterson, T. Rohlfing, J. Hall, R. Brown, N. Lakey, R.W. Doerge, R.A. Martienssen, J. Leon, J.D. McPherson, and J.A. Jeddeloh. 2007. Identification of novel high-frequency DNA methylation changes in breast cancer. Public Library of Science (PLoS) ONE. 12:1314.
36. R.W. Doerge. 2007. Proper experimental design and sound statistical inference win every time: a commentary on &#39;Statistical design and the analysis of gene expression microarray data&#39; by M. Kathleen Kerr and Gary A. Churchill. Genetics Research. 89(5-6): 505-507.
37. H.-T. Leung, L. An\*, J. Tseng-Crank, E. Kim, E.L. Harness, Y. Zhou, J. Kitamoto, G. Li, R.W. Doerge, and W.L. Pak. 2008. Phototransduction in Drosophila: Use of microarrays in cloning genes identified by chemically induced mutations causing ERG defects. In S.J. Fliesler and O.G. Kisselev (eds.), Signal Transduction in the Retina, Taylor and Francis/CRC Press, ppl95-217.
38. M. Bogdan, P. Biecek, R. Cheng\*, F. Frömmlet, J.K. Ghosh, and R.W. Doerge. 2008. Extending the modified Bayesian Information Criterion (mBIC) to dense markers and multiple interval mapping. Biometrics. 64(4):1162-1169.
39. A.A. Long, E. Kim, H.-T. Leung, E. Woodruff HI, L. An\*, R.W. Doerge, W.L. Pak, and K. Broadie. 2008. Presynaptic calcium channel localization and calcium-dependent synaptic vesicle exocytosis regulated by the fuseless protien. Journal of Neuroscience. 28:3668 - 3682.
40. G.A. Churchill and R.W. Doerge. 2008. Naive application of permutation testing leads to inflated type I error rates. Genetics. 178:609-610.
41. H. Jiang\* and R.W. Doerge. 2008. Estimating the proportion of true null hypotheses for multiple comparisons. Cancer Informatics. 4:25-32.
42. H.-T. Leung, J. Tseng-Crank, E. Kim, C. Mahapatra, S. Shino, Y. Zhou, L. An, R.W. Doerge and W.L. Pak. 2008. DAG lipase activity is necessary for TRP channel regulation in Drosophila photoreceptors. Neuron. 58(6):884-896.
43. M. Tanurdzic, M. Vaughn, H. Jiang\*, L. Lee, E. Thompson, R.W. Doerge, and R.A. Martienssen. 2008. Epigenomic consequences of immortalized plant cell suspension culture. Public Library of Science Biology (PLoS Biology). 6(12):2880-2895.
44. S.-Y. Yoo\* and R.W. Doerge. Integrating Epigenomics Results. 2008. Epigenomics.

A. Ferguson-Smith et al. (eds) Springer-Verlag. 37-52.

1. R.W. Doerge. 2008. Statistical Advances in Functional Genomics. R. Varshney and R. Tuberosa (eds.), Genomic Assisted Crop Improvement: Vol 1: Genomics Approaches and Platforms, 315-332. Springer-Verlag.
2. K. Spach, L. An\*, M. Blake, E. Blankenhorn, J. Bunn, R. Doerge, B. McElvany, R. Noubade, K. Tung, and C. Teuscher. 2008. Age, season and stress influence EAE sexual dimorphism in SJL/J mice. Journal of Neuroimmunology. 203(2):212-212.
3. P. Jones, T. Archer, S. Baylin, et al. 2008. Moving AHEAD with an international human epigenome project . Nature. 454(7205):711-715.
4. L. An\* and R.W. Doerge. 2008. Dynamic clustering of cell-cycle gene expression data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp!8-36.
5. S.-Y. Yoo\* and R.W. Doerge. 2008. A two-stage approach for estimating the effect of DNA methylation on differential expression using tiling array technology. 2008. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp!24-137.
6. R.T. Gaeta, S.-Y. Yoo\*, J.C. Pires, R.W. Doerge, Z.J. Chen, and T.C. Osborn. 2009. Analysis of gene expression in resynthesized Brassica napus allopolyploids using Ara- bidopsis 70mer oligo microarrays. Public Library of Science (PLoS) ONE 4(3): e4760. doi:10.1371/journal.pone.0004760 .
7. M. Bremer\* and R.W. Doerge. 2009. The KM-Algorithm identifies genes in time series gene expression data. Advances in Bioinformatics. Volume 2009, Article ID 284251, 10 pages, doi:10.1155/2009/284251.
8. N. C. Riddle, H. Jiang\*, L. An\*, R.W. Doerge, and J.A. Birchler. 2009. Gene expression analysis at the intersection of ploidy and hybridity in maize. Theoretical and Applied Genetics. 120(2):341-353.
9. R.W. Doerge and K. Kim\*. 2009. Statistical challenges in modeling expression quanti¬tative traits. Proceedings of the 24th International Workshop on Statistical Modelling. James E. Booth, Editor, ppl3-22.
10. G.R. Olbricht\*, N. Sardesai, S.B. Gelvin, B.A. Craig, and R.W. Doerge. 2009. Statis¬tical methods for Affymetrix tiling array data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pplll-125.
11. P. L. Auer\* and R.W. Doerge. 2009. Statistical issues in next generation sequencing. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp69-84.
12. A. Lipka\*, G. McCabe and R.W. Doerge. 2009. Associating SNPs with binary traits. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp5-19.
13. C. Ochsenfeld\*, K. Jennings, and R.W. Doerge. 2009. Mapping QTL with covariates. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. ppl26-140.
14. D. Pignatta, B.P. Dilkes, S.-Y. Yoo\*, I.M. Henry, A. Madlung, R.W. Doerge, Z. J. Chen, L. Comai. 2010. Differential sensitivity of the Arabidopsis thaliana transcriptome and enhancers to the effects of genome doubling. New Phytologist. 186(l):194-206.
15. A. Rau\*, F. Jaffrezic, J.-L. Foulley, and R.W. Doerge. 2010. An empirical Bayesian method for estimating biological networks from temporal microarray data. Statistical Applications in Molecular Biology and Genetics. 9(1) Article 9.
16. M. Bremer\* and R.W. Doerge. 2010. Statistics at the Bench: A Step-by-Step Hand¬book for Biologists. Cold Spring Harbor Press. Cold Spring Harbor, NY.
17. P.L. Auer\* and R.W. Doerge. 2010. Statistical design and analysis of RNA sequencing data. Genetics. 185: 405-416.
18. A.A. Long, C. Mahapatra, H.-T. Leung, E.A. Woodruff HI, J. Rohrbough, L. An\*, R.W. Doerge, M.M. Metzstein, S. Shino, W.L. Pak, and K. Broadie. 2010. The nonsense-mediated decay pathway is critical for maintaining synapse architecture and synaptic vesicle cycle efficacy. Journal of Cell Science. 3303-3315.
19. L. Yu\* and R.W. Doerge. 2010. A semi-non-parametric mixture model for selecting functionally consistent proteins. BMC Bioinformatics. 11:486 doi: 10.1186/1471-2105¬11-486.
20. P.L. Auer\* and R.W. Doerge. 2010. Functional divergence of duplicated genes in the soybean genome. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp!71-185.
21. A. Rau\*, F. Jaffrezic, J.-L. Foulley, and R.W. Doerge. 2010. Approximate Bayesian approaches for reverse engineering biological networks. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp202-216.
22. G. Olbricht\*, B.A. Craig, and R.W. Doerge. 2010. Modeling DNA methylation tiling array data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp63-84.
23. S. Srivastava\* and R.W. Doerge. 2010. A non-parametric empirical Bayes approach for estimating transcript abundance in unreplicated next-generation sequencing data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp230-244.
24. T. Achberger\*, J.C. Fleet, D.E. Salt, and R.W. Doerge. 2010. Introduction to selecting subsets of traits for quantitative trait loci analysis. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp267-286.
25. P.L. Auer\* and R.W. Doerge. 2011. A two-stage Poisson model for testing RNA-seq data. Statistical Applications in Genetics and Molecular Biology. 10(1) Article 26.
26. D. Baumann\* and R.W. Doerge. 2011. Issues in testing DNA methylation using next¬generation sequencing. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. ppl-14.
27. S. Sri vast ava\* and R.W. Doerge. 2011. A hierarchical Bayesian approach for detecting differential gene expression in unreplicated RNA-sequencing data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhat¬tan, KS. pp!66-189.
28. P.L. Auer\*, S. Srivastava\*, and R.W. Doerge. 2012. Differential Expression: The next generation and beyond. Briefings in Functional Genomics. ll(l):57-62.
29. S. Srivastava\* and R.W. Doerge. 2012. Purdue Statistics: A journey through time. &quot;Strength in Numbers: The Rising of Academic Statistics Departments in the United States&quot;. Editors Alan Agresti and Xiao-Li Meng. Springer. pp229-242.
30. G. Olbricht\*, B. Craig, and R.W. Doerge. 2012. Incorporating genomic annotation into a hidden markov model for DNA methylation tiling array data. Statistical Appli¬cations in Genetics and Molecular Biology. 11(5) DOI: 10.1515/1544-6115.1775.
31. A. Rau\*, F. Jaffrezic, J.-L. Foulley and R. W. Doerge. 2012. Reverse engineering gene regulatory networks using approximate Bayesian computation. Statistics and Comput¬ing: 22(6):1257-127.
32. L. An\* and R.W. Doerge. 2012. Dynamic clustering of gene expression. ISRN Bioinformatics, doi:10.5402/2012/537217.
33. A.A. Elias\*, K.R. Robbins, D. Niyogi, J.J. Camberato, R.W. Doerge, and M.R. Tu- instra. 2012. Evaluation of genotype by environment interactions from unreplicated multi-environmental trials of hybrid maize. The Proceedings of the Kansas State Uni¬versity Conference on Applied Statistics in Agriculture. Manhattan, KS. ppl-12.
34. D. Baumann\* and R.W. Doerge. 2012. Correcting for amplification bias in next¬generation sequencing data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp36-49.
35. X. Chen\* and R.W. Doerge. 2012. Towards better FDR procedures for discrete test statistics. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp294-303.
36. S. Srivastava\* and R.W. Doerge. 2012. The nuances of statistically analyzing next¬generation sequencing data. The Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture. Manhattan, KS. pp233-267.
37. A. Roulin, P.L. Auer\*, M. Libault, J. Schlueter, A. Farmer, G. May, G. Stacey, R.W. Doerge and S.A. Jackson. 2013. The fate of duplicated genes in a polyploid plant genome. The Plant Journal. 73:143153.
38. C.N. Billingsley, J.A. Allen, J.D. Blazek, D.D. Baumann\*, A. Newbauer, A. Darrah, M. Clement, R.W. Doerge, and R.J. Roper. 2013. Non-trisomic homeobox gene expression during craniofacial development in Down syndrome mice. American Journal of Medical Genetics. 161(8):1866-1874.
39. R. Cheng\*, J. Borevitz, and R.W. Doerge. 2013. Selecting informative traits for multivariate quantitative trait locus mapping helps to gain optimal power. Genetics. 195(3):683-691.
40. N. Sardesai, H. Chen, H. Yi, G.R. Olbricht\*, K. Laluk, R.W. Doerge, T. Mengiste, and S.B. Gelvin. Cytokinins secreted by agrobacterium promote transformation by repressing a plant Myb transcription factor. Science Signaling 6 (302):ral00.
41. D.D. Baumann and R.W. Doerge. 2014. Robust adjustment of sequence tag abun¬dance. Bioinformatics. 30(5):601-605.
42. I. Pardo, H.A. Lillemoe, R.J. Blosser, M. Choi, C.A.M. Sauder, D.K. Doxey, T. Math- ieson, B.A. Hancock, D. Baptiste, R. At ale, M. Hickenbotham, J. Zhu, J. Glasscock, A.M.V. Storniolo, F. Zheng, R.W. Doerge, Y. Liu, S. Badve, M. Radovich, and S.E. Clare. 2014. Next-generation transcriptome sequencing of the premenopausal breast epithelium using specimens from a normal human breast tissue bank. Breast Cancer Research. 16(2) :R26
43. F. Lin, M. Zhao, D.D. Baumann, J. Ping, L. Sun, Y. Liu, B. Zhang, Z. Tang, E. Hughes, R.W. Doerge, T.J. Hughes and J. Ma. 2014. Molecular response to the pathogen phytophthora sojae among ten soybean near isogenic lines revealed by comparative transcriptomics. BMC Genomics. 15(1):18
44. D.D. Baumann and R.W. Doerge. 2014. MAGI: Methylation analysis using genome information. Epigenetics. 9(5):0-l.
45. M. Bremer\* and R.W. Doerge. 2015. Using R at the Bench: Step-by-Step Analytics for Biologists. Cold Spring Harbor Press. Cold Spring Harbor, NY.

**Manuscripts/Chapters Submitted for Publication (8)**

Current and former Purdue graduate students denoted by \*

1. M. Tanurdzic, P. L. Auer\*, P. Finigan, E. Ernst, B. Dilkes, L. Comai, B. Meyers, M.

Vaughn, R.W. Doerge, R. Martienssen. Transposon instability in inter-species hybrids is mediated by RNA interference. Science (under revision).

1. J. Cardenas-Mora, C. Plachot, L. An\*, D. Baumann\*, J. Stanley\*, H. Adissu, A. Schafer,

A. Lipka\*, K. Hodges, S. Badve, R.W. Doerge, S. Lelievre. Disruption of tissue polarity modifies the expression of key genes for breast homeostasis and identifies phospholipid synthase, CDS1, as a new candidate in breast cancer. Journal of Pathology.

1. S. Srivastava\* and R.W. Doerge. 2015. Latent process decomposition of high-dimensional

count data. Biostatistics.

1. S. Srivastava\* and R.W. Doerge. 2015. Dynamic latent process decomposition of genomic

time-course data. Biostatistics.

1. A.A. Elias\*, K.R. Robbins, R.W. Doerge, and M. R. Tuinstra. Half a century of studying

genotype by environment interactions in plant breeding experiments. Crop Science.

1. X. Chen\* and R.W. Doerge. 2015. Estimating proportion of nonzero means under certain strong covariance dependence. Electronic Journal of Statistics.
2. X. Chen\* and R.W. Doerge. 2015. Generalized estimators for multiple testing: propor¬

tion of true nulls and false discovery rate. Electronic Journal of Statistics.

1. J.H. Oh\*, F. Zheng\*, R.W. Doerge, H. Chun. 2015. Kernel partial correlation: A novel

approach to capturing conditional independence in graphical models for noisy data. Computational Statistics and Data Analysis.

**Manuscripts, Book Chapters, and Books in Progress (4)**

Current and former Purdue graduate students denoted by \*

1. C. Ochsenfeld\*, K. Jennings, and R.W. Doerge. Bootstrap thresholds for mixed model

QTL and association mapping. Genetics.

1. C. Ochsenfeld\*, K. Jennings, and R.W. Doerge. Mixed models for QTL mapping.
2. M. Bogdan, D.O. Siegmund, P.A. Szulc, H. Tang. 2013. Are &quot;Hot-spots&quot; an indication

of complex interactions?

1. F. Zheng\* and R.W. Doerge. Statistical considerations of single-cell sequencing experi¬

ments.

**Published Abstracts, Non-Refereed Publications, and Conference Pro¬ceedings (43)**

Current and Purdue graduate students denoted by \*

1. R.W. Doerge. 1988. _Information of Fixed Cluster Samples for Genetic Traits Involving_

_Mixtures of Distributions._ Master of Statistics in Mathematics, University of Utah.

1. M.M. Shoukri, R.H. Ward, R.W. Doerge. 1989. Statistical inference from the genetic

mixture of normal distributions, Proceedings of the 47th Session of the International Statistical Institute.

1. R.D. Sederoff, D. Grattapaglia, P. Wilcox, J. Chaparro, D. O&#39;Malley, S. McCord, R.

Whetten, L. McIntyre, B. Weir, R.W. Doerge, F. Bridge water, S. Thorpe. 1992. Use of PCR-based RAPD markers for genetic mapping in conifers. Abstracts of papers of the American Chemical Society. 203: 73-BTEC, Part 3.

1. B.S. Weir and R.W. Doerge. 1992. Statistical issues in the search for QTLs. XVI International Biometric Conference Proceedings, Contributed Papers p. 164.
2. B.S. Weir and R.W. Doerge. 1993. Biometrical issues in the search for genes affecting

quantitative traits. American Statistical Association Annual Meeting Abstracts p311.

1. R.W. Doerge. 1993. _Statistical Methods for Locating Quantitative Trait Loci with Molec¬_

_ular Markers._ Ph.D. Dissertation. Department of Statistics. North Carolina State University, Raleigh, NC.

1. G.A. Churchill and R.W. Doerge. 1994. Mapping quantitative trait loci in experimental

populations. Proceedings of the Biometric Conference. Ontario, Canada.

1. R.W. Doerge and G.A. Churchill. 1994. Analysis of molecular marker data. Issues in

genetic mapping of quantitative trait loci. Proceedings of the ASHA/CSSA 2nd Joint Plant Breeding Symposium. Corvallis, Oregon.

1. R.W. Doerge, Z-B. Zeng and B.S. Weir. 1994. Statistical issues in the search for genes

affecting quantitative traits in populations. Proceedings of the ASHA/CSSA 2nd Joint Plant Breeding Symposium. Corvallis, Oregon.

1. R.W. Doerge. 1996. Locating genes in plants, animals and humans using Statistics and Genetics. Sequel. Purdue University School of Science publication.
2. R.W. Doerge. 1998. Characterizing germplasm collections using molecular markers. Abstracts Proceedings of the American Statistical Association.
3. R.W. Doerge. 1999. Risk communication for locating quantitative trait loci (QTL) in experimental populations. Abstracts Proceedings of the American Statistical Associa-

tion.

1. S.M. Brouder, C.H. Nakatsu and R.W. Doerge. 1999. Microbial community dynamics in the rhizosphere of Zea mays L. and Glycine max. Amer. Soc. Agronomy, Salt Lake City, UT (Oct. 31 - Nov. 5, 1999).
2. C.H. Nakatsu, S.M. Brouder, F. Wanjau, J.D. Wilbur\*, and R.W. Doerge. 2000. Impact of tillage and crop rotation on corn development and its associated microbial commu¬nity. Proceedings of the 15th Conference of the International Soil Tillage Research Organization (ISTRO). July 2-7, 2000. Fort Worth, TX (CD-ROM).
3. J.D. Wilbur\*, S.M. Brouder, C.H. Nakatsu, and R.W. Doerge. 2000. Statistical analysis of rhizosphere microbial communities. In Proceedings of Interface 2000. The 32nd Symposium on the Interface: Computing science and statistics. New Orleans, LA (CD¬ROM).
4. F.M Wanjau, S.M Brouder, C.H. Nakatsu, and R.W. Doerge. 2000. Influence of crop rotation and tillage practices on root architecture of corn and soybean. American Society of Agronomy Abstracts. Minneapolis, MN.
5. R.J. Roper, R.W. Doerge, K.S.K. Tung, C. Teuscher. 2000. Genetic loci controlling thymectomy-induced autoimmune ovarian dysgenesis. NIH Workshop on The Ovary, Gensis, Function and Failure (Bethesda, MD).
6. M.A. Black\*, B.A. Craig, M.J. Tanurdzic\*, and R.W. Doerge. 2001. A Bayesian approach to the analysis of cDNA microarray data. Proceedings of the 33rd Symposium on the Interface.
7. D.A. Bulinski\*, C.H. Nakatsu, J.D. Wilbur\*, S. M. Brouder, and R.W. Doerge. 2001. Rhizosphere Microbial population dynamics over three field seasons. American Society of Agronomy Abstracts. Charlotte, NG.
8. H.-S. Lee, J. Wang, L. Tian, L. Lukens, A. Madlung, M.A. Black\*, L. Comai and R.W. Doerge. 2002. Genome-wide expression analysis of orthologous genes in _Arabidopsis_ and _Brassica_ polyploids. Plant Biology 2002. Denver, CO.
9. T.C. Osborn, L. Lukens, J.C. Pires, Z.J. Chen, L. Comai, R.W. Doerge, R.A. Mar- tienssen. 2002. Why become a polyploid? Plant and Animal Genome X, p65.
10. T.C. Osborn, L. Lukens, J.C. Pires, M.E. Schranz, H.-S. Lee, Z.J. Chen, R.W. Doerge. 2002. Gene expression studies in Brassica. 13&#39;&quot; Crucifer Genet Workshop Abstracts, p7L
11. M.A. Black\*, B.A. Craig, J. Deely, R.W. Doerge. 2002. Bayesian analysis of cDNA microarray data. Joint Statistical Meetings, p271.
12. J.D. Wilbur\*, J.K. Ghosh, C.H. Nakatsu, S.M. Brouder, R.W. Doerge. 2002. An appli¬cation of empirical bayes model selection to microbial community fingerprint analysis. Joint Statistical Meetings, p272.
13. A. Madlung, B. Watson, R. Masuelli, Z. Lippman, V. Colot, R. Martienssen, R.W. Doerge, M. Black, L. Comai, L. 2003. Genetic and epigenetic changes in synthetic allopolyploids of _Arabidopsis thaliana._ Plant and Animal Genome XI, p63.
14. D. Cao\*, B.A. Craig, R.W. Doerge. 2003. A maximum likelihood-based interval map¬ping model for autopolyploids. Joint Statistical Meetings, p27.
15. K. Kim\* and R.W. Doerge. 2003. Statistical issues in dissecting regulatory networks associated with complex traits: genetic mapping of gene expression levels. Joint Sta¬tistical Meetings, p27.
16. B.A. Craig and R.W. Doerge. 2003. Gene expression data: The technology and statistical analysis. Joint Statistical Meetings, p67.
17. A. Madlung, B. Watson, H. Jiang\*, T. Kagochi, R.W. Doerge, L. Comai, R. Mar-tienssen. 2004. Transposon activation in Arabidopsis neopolyploids. Arabidopsis Meet¬ing, Berlin, Germany.
18. J. Chen, J. Wang, L. Tian, E. Weil, M. Chen, S. Rao, R.W. Doerge, H. Jiang\*, L. Comai, B. Watson. 2005. Transcriptome divergence and mechanisms of non-additive gene regulation in Arabidopsis allopolyploids. International Botanical Congress.
19. M. Bogdan, J. K. Ghosh, R.W. Doerge, P. Biecek, A. Baierl, A. Futschik, F. Frömmlet. 2005. Modified version of Schwarz Bayesian Information Criterion for localization of multiple interacting quantitative trait loci. Paris, France. Annals of Human Genetics.
20. M. Bogdan, J.K. Ghosh, R.W. Doerge, P. Biecek, A. Baierl, A. Futschik, F. Frömmlet. 2005. Modified version of Bayesian Information criterion for localization of multiple interacting quantitative trait loci. Ann. Hum. Gen. 69:765.
21. M. Vaughn, M. Tanurdzic, Z. Lippman, H. Jiang\*, V. Colot, R.W. Doerge, and R. Martienssen. 2005. Epigenomic profiling of Arabidopsis Thaliana. Cold Spring Harbor Laboratory Genome Informatics Meeting. Cold Spring Harbor, NY.
22. R.A. Martienssen, M. Zaratiegui, D. Irvine, D. Goto, K. Slot kin, M. Tanurdzic, Z. Lippman, M. Vaughn, B. May, Z. Cande, V. Colot, and R.W. Doerge. 2007. Silent running: RNA interference and heterochromatic modification. Chromosome Research 15:20-21 Supplement 2.
23. R.A. Martienssen, M. Tanurdzic, K. Slotkin, P. Finigan, K. Creasey, P. Auer\*, M. Vaughn, J.-P. Vielle-Calzada, and R.W. Doerge. 2011. Heterochromatin reprogram-ming by small RNA regulates germ cell fate in Arabidopsis. Plant Epigenome Work-

shop. Plant and Animal Genome Conference. San Diego, GA.

1. J.M. Cardenas-Mora, A. Schafer, C. Plachot, J. Stanley, D. Stegelman, A. Lipka\*, D. Baumann\*, K. Hodges, K. McDole, L. An\*, R.W. Doerge, and S.A. Lelievre. 2011. CDS1, a potential novel marker of breast cancer development. 3rd Annual National Predoctoral Clinical Research Training Program Meeting. St Louis, MO.
2. S. Chudalyandi, R.W. Doerge, S. Srivastava\*, and J. Birchler. 2011. Gene expression profiling in normal and interploidy endosperms of maize. ASPB Plant Biology Meeting.
3. M. Tanurdzi, P. Auer\*, P. Finigan, E. Ernst, B. Dilkes, L. Comai, B.C. Meyers, M.W. Vaughn, R.W. Doerge, R.A. Martienssen. 2011. Small RNA mediate transposon in-stability in plant inter-specific hybrids in a parent-specific manner. European Science Foundation (ESF) European Molecular Biology Organization (EMBO) &quot;Epigenetics in Context: From Ecology to Evolution&quot;. Sant Feliu de Guixols, Spain.
4. R.J. Roper, C. Billingsley, J. Allen, J. Blazek, D. Baumann, A, Newbauer, A. Darrah 1, B.C. Long, B. Young, M. Clement, and R.W. Doerge. 2011. Non-trisomic homebox gene AND sox9 expression alters craniofacial development in a down syndrome mouse model. Mouse Genetics Conference. Washington, D.C.
5. A.A. Elias, K.R. Robbins, Dev Niyogi, James J. Camberato, R.W. Doerge, and M.R. Tuinstra. 2011. Estimation and prediction of hybrid maize performance in mega-environments and changing climate American Society of Agronomy. San Antonio, TX.
6. R.A. Martienssen, M. Tanurdzic, P. Baduel, J. Calarco, K. Creasey, F. vanEx, Y. Jacob, P.L. Auer, J.-P. Vielle-Calzada, and R.W Doerge. 2011. RNA interference, heterochromatin reprogramming and germ cell fate. Conference on epigenetics and the inheritance of acquired states. Boston, MA.
7. R.A. Martienssen, M. Tanurdzic, P.L. Auer, P.E. van Ernst, B.P. Dilkes, L. Comai,B.C. Meyers, M. Vaughn, R.W. Doerge. 2012. Transposon activation in Arabidopsis al¬lopolyploids is mediated by RNA interference. Plant and Animal Genome XX, San Diego, CA.
8. R.W. Doerge. 2012. Statistical issues and the analysis of next-generation sequencing data. Plant and Animal Genome XX, San Diego, CA.



# Information Reference
  - homepage: http://www.stat.cmu.edu/people/faculty/rebecca-doerge 
  - homepage: https://www.cmu.edu/mcs/people/dean-bio.html
  - googlescholar: https://scholar.google.com/citations?user=B4WVzSEAAAAJ&hl=en  
  - aminer: https://www.aminer.cn/profile/rebecca-doerge/5605be0145ce1e595e7cc95d

# Notes