---
bio-current:
  name-cn: 刘晓乐
  name-en: Xiaole Liu
  email: 
    - xsliu@jimmy.harvard.edu # 如果有多个邮箱，请都填写上
  sex: female
  birth-date: # yyyy 到年即可
  university: Harvard University 
  school: Dept, of Biostatistics [https://www.hsph.harvard.edu/biostatistics] # 格式：学院名称[学院官网链接]
  major: Biostatistics
  title-raw: Professor # 主页原始字符串
  title: Professor
  interests: 
    - omputational models of transcription
    - epigenetic regulation by integrating data from genome-wide ChIP-chip / ChIP-Seq # 分点罗列，依次以 ‘-’ 开头
  homepage: 
    - https://www.hsph.harvard.edu/xiaole-liu/
    - https://liulab-dfci.github.io/ # 如果有多个主页，请都填写上
  github: 
  googlescholar: https://scholar.google.com/citations?hl=zh-CN&user=8XNfVucAAAAJ 
  aminer: https://www.aminer.cn/profile/xiaole-liu/560fffca45ce1e5961fa5add # 从这里查找 https://www.aminer.org/search/person
  status: 在职
  last-update: # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: Stanford University
  school: Biomedical Informatics and Minor Computer Science
  email: 
  date-start: 
  date-end: 2002
  advisor: # 格式：导师名 [邮箱/网址]
  degree: # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: Smith College
  school: Biochemistry and Computer Science
  date-start: 
  date-end: 1997
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: 
  research: 
  software: 
  project: 
  blog: 
  arxiv: 
  linkedin: 
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: Harvard University 
  school: Dept, of Biostatistics [https://www.hsph.harvard.edu/biostatistics] # 格式：学院名称[学院官网链接]
  major: Biostatistics 
  email: xsliu@jimmy.harvard.edu
  homepage: https://www.hsph.harvard.edu/xiaole-liu/ # 机构内学者主页
  date-start: 
  title: Professer
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Xiaole Liu](//avatarcdn.aminer.cn/picture/extend_data/cj/2011/images/2011211.jpg!240?ran=0.12921082705447495)

# Biography[English]

My research focuses on computational models of transcription and epigenetic regulation by integrating data from genome-wide ChIP-chip / ChIP-Seq, nucleosome occupancy and histone modifications, gene expression microarray / RNA-sequencing, genomic sequence and conservation. To this end, we focus on the following areas of studies:

First, we developed a number of algorithms widely used and cited for transcription factor motif discovery from promoters of gene expression clusters (BioProspector, Liu et al, PSB2001, cited by 419), ChIP-chip (MDscan, Liu et al, Nature Biotech 2002), gene expression perturbation experiments (Motif Regressor, Conlon et al, PNAS 2003), and using comparative genomics (CompareProspector, Liu et al, Genome Res 2004). These tools are also available as a web server (Liu et al, NAR 2004). We are currently developing a tool to find sequence motifs from ChIP-seq and nucleosome sequencing.

Second, we design computational algorithm for ChIP-chip and ChIP-seq data analysis. We published the 3rd ever ChIP-chip paper and the first to integrate ChIP-chip with transcription factor motif discovery (Lieb et al, Nat Genet 2001). We also published the 2nd ever genome-wide ChIP-chip study in human, which is the first to systematically study genome-wide enhancer binding function (Carroll et al, Nat Genet 2006). We developed widely used ChIP-chip (MAT, Johnson et al, PNAS 2006; MA2C, Song et al, Genome Biol 2007) and ChIP-seq analysis algorithms (MACS, Zhang et al, Genome Biol 2008), and downstream analysis pipeline (CEAS, Ji et al, NAR 2006). I also coordinated a study in the NIH ENCODE consortium (involving all the ChIP-chip pioneers) to systematically evaluate the effect of DNA amplification method, tiling array platforms and analysis algorithms on ChIP-chip results (Johnson et al, Genome Res 2008). I am currently organizing a similar study in the NIH mod/ENCODE consortiums comparing different algorithms and high throughput sequencing platforms.

My third area of interest is on epigenetic regulation. We published the 1st high throughput nucleosome positioning study in human (Oszolak et al, Nat Biotech 2007). We also designed algorithms to identify positioned nucleosomes from nucleosome-resolution histone modification ChIP-seq data (Zhang et al, BMC Genomics, 2008) and use nucleosome positioning and histone modification to predict miRNA promoters (Oszolak et al, Gene Dev, in press). I am currently working on finding differential nucleosome positions between different conditions, and comparing the intrinsic nucleosome positioning with in vivo positioning. I am also a co-PI for the worm chromatin group at the NIH modENCODE consortium.

Finally, we integrate the tools and data for ChIP-chip/seq to understandings to study trasncription and epigenetic regulatory network in cancer, aging, diabetes and stem cell differentiation. Through genome-wide ChIP-chip analysis, we discovered important transcriptional and epigenetic regulation mechanisms of estrogen receptor (ER) in breast cancer (Carroll et al, Cell 2005; Carroll et al, Nat Genet 2006), androgen receptor (AR) in prostate cancer (Wang et al, Mol Cell 2006), ER and AR’s collaborating transcription factor and chromatin remodeler FoxA1 in breast and prostate cancers (Lupien et al, Cell 2008), and peroxisome proliferator-activated receptor gamma (PPAR?) in adipose tissues (Lefterova et al, Gene Dev 2008). We are currently directing bench and computational experiments to identify the epigenetic signature and infer the transcription regulatory network of hormone independent breast and prostate cancers.

# Biography[中文]

# Interests[English]

* omputational models of transcription
* epigenetic regulation by integrating data from genome-wide ChIP-chip / ChIP-Seq

# Interests[中文]

# Education[English]

- BA in Biochemistry and Computer Science, 1997
    
    Smith College
    
- PhD in Biomedical Informatics and Minor Computer Science, 2002
    
    Stanford University
    
# Education[中文]

# Awards[English]

# Awards[中文]

# Talks[English]

# Talks[中文]

# Work experience[English]

# Work experience[中文]

# Publication[English]

# Publication[中文]

# Information Reference

-  homepage: https://www.hsph.harvard.edu/xiaole-liu/
-  googlescholar: https://scholar.google.com/citations?hl=zh-CN&user=8XNfVucAAAAJ
-  aminer: https://www.aminer.cn/profile/xiaole-liu/560fffca45ce1e5961fa5add

# Notes