---
bio-current:
  name-cn: 李克赫
  name-en: Kyu Ha Lee
  email: 
    - klee@hsph.harvard.edu # 如果有多个邮箱，请都填写上
  sex: male
  birth-date: # yyyy 到年即可
  university: Harvard University 
  school: Dept, of Biostatistics [https://www.hsph.harvard.edu/biostatistics/] # 格式：学院名称[学院官网链接]
  major: Epidemiology and Nutrition
  title-raw: Assistant Professor # 主页原始字符串
  title: Assistant Professor
  interests: 
    - Bayesian analysis
    - Multivariate analysis
    - Survival analysis
    - Microbiome data analysis
    - High-dimensional data analysis # 分点罗列，依次以 ‘-’ 开头
  homepage: 
    - http://www.hsph.harvard.edu/kyu-ha-lee 
    - https://sites.google.com/site/kyuhalee000/ # 如果有多个主页，请都填写上
  github: 
  googlescholar: https://scholar.google.com/citations?hl=zh-CN&user=6cxMccMAAAAJ
  aminer: None # 从这里查找 https://www.aminer.org/search/person
  status: 在职
  last-update: 2021-04-29 # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: University of Missouri
  school: Statistics
  email: 
  date-start: 8/2006
  date-end: 7/2011
  advisor: # 格式：导师名 [邮箱/网址]
  degree: # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: Harvard School of Public Health
  school: Biostatistics
  date-start: 8/2011
  date-end: 12/2013
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: https://connects.catalyst.harvard.edu/Profiles/display/Person/102364
  research: https://connects.catalyst.harvard.edu/Profiles/display/Person/102364
  software: 
  project: 
  blog: 
  arxiv: 
  linkedin: 
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: Harvard University 
  school: Dept, of Biostatistics [https://www.hsph.harvard.edu/biostatistics/] 
  major: Epidemiology and Nutrition 
  email: klee@hsph.harvard.edu
  homepage: http://www.hsph.harvard.edu/kyu-ha-lee # 机构内学者主页
  date-start: 
  title: Assistant Professor
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Kyu Ha Lee](https://connects.catalyst.harvard.edu/Profiles/profile/Modules/CustomViewPersonGeneralInfo/PhotoHandler.ashx?NodeID=62721865)

# Biography[English]

I am interested in developing statistical methods for multivariate and/or high-dimensional biomedical data from a wide range of applications:

1. Multivariate statistical methods for microbiome data. One primary focus of my research program is on statistical methods development for microbiome study. One research goal is to develop spatial point pattern analysis methods for understanding the spatial organization of microbes by using spectral imaging data. Another research goal is to develop comprehensive multivariate methods for microbiome sequencing count data. These methods differ from most commonly used techniques in that they involve analyzing the spatial/counts distributions of all microbial types as a joint endpoint distribution, instead of analyzing the univariate distribution of each type separately (taxon-by-taxon analysis). The overarching goal is to provide more robust and valid quantitative analysis tools to scientists in microbiology and bioinformatics.

2. Semi-competing risks framework for multivariate survival data. Semi-competing risks refers to the setting where interest lies in a nonterminal event (e.g. hospital readmission), the occurrence of which is subject to a terminal event (e.g. death). Although less known than competing risks, semi-competing risks problem arises in a broad range of public health applications. I have developed a novel hierarchical modeling framework for the analysis of clustered semi-competing risks survival data. The framework permits parametric or nonparametric specifications for a range of model components, including baseline hazard functions and distributions for key random effects, giving analysts substantial flexibility as they consider their own analyses. I am currently extending the method for various type of study designs to further expand the scope of scientific inquiry from clinical and public health science.

3. Survival analysis with high-dimensional genomic covariates. Developing a predictive model that relates the time-to-event outcome to high-dimensional genomic data is challenging because of i) high-dimensional genomic variables, the number of which often far exceeds the number of subjects, ii) structured grouping of genes, and iii) censored outcomes. I am developing statistical methods for correlated and structured high-dimensional genomic data with survival outcomes in the context of penalized regression models.

# Biography[中文]

# Interests[English]

* Bayesian analysis
* Multivariate analysis
* Survival analysis
* Microbiome data analysis
* High-dimensional data analysis

# Interests[中文]

# Education[English]

- PhD in Statistics, 8/2006 - 7/2011
    
    University of Missouri, Columbia, MO
    
- Postdoctoral Research Fellow in Biostatistics, 8/2011 - 12/2013
    
    Harvard School of Public Health, MA
    
# Education[中文]

# Awards[English]

# Awards[中文]

# Talks[English]

# Talks[中文]

# Work experience[English]

# Work experience[中文]

# Publication[English]

- Li J, Li Y, Ivey KL, Wang DD, Wilkinson JE, Franke A, **Lee KH**, Chan A, Huttenhower C, Hu FB, Rimm EB, Sun Q. Interplay between diet and gut microbiome, and circulating concentrations of trimethylamine N-oxide: findings from a longitudinal cohort of US men. Gut. 2021 Apr 29. PMID: 33926968.
- Li Y, Seo S, **Lee KH**. Bayesian analysis of grouped survival data with adaptive time partition. J Stat Comput Simul. 2021; in press. View Publication.
- **Lee KH**, Coull BA, Moscicki AB, Paster BJ, Starr JR. Bayesian variable selection for multivariate zero-inflated models: Application to microbiome count data. Biostatistics. 2020 07 01; 21(3):499-517. PMID: 30590511.
- Goldstein JM, Valido A, Lewandowski JP, Walker RG, Mills MJ, Messemer KA, Besseling P, **Lee KH**, Wattrus SJ, Cho M, Lee RT, Wagers AJ. Variation in zygotic CRISPR/Cas9 gene editing outcomes generates novel reporter and deletion alleles at the Gdf11 locus. Sci Rep. 2019 12 09; 9(1):18613. PMID: 31819086.
- Kim JH, Pegoraro AF, Das A, Koehler SA, Ujwary SA, Lan B, Mitchel JA, Atia L, He S, Wang K, Bi D, Zaman MH, Park JA, Butler JP, **Lee KH**, Starr JR, Fredberg JJ. Unjamming and collective migration in MCF10A breast cancer cell lines. Biochem Biophys Res Commun. 2020 01 15; 521(3):706-715. PMID: 31699371.
- Alvares D, Haneuse S, Lee C, **Lee KH**. SemiCompRisks: An R Package for the Analysis of Independent and Cluster-correlated Semi-competing Risks Data. R J. 2019 Jun; 11(1):376-400. PMID: 33604061.
- Green DR, Schulte F, **Lee KH**, Pugach MK, Hardt M, Bidlack FB. Mapping the Tooth Enamel Proteome and Amelogenin Phosphorylation Onto Mineralizing Porcine Tooth Crowns. Front Physiol. 2019; 10:925. PMID: 31417410.
- Bassir SH, El Kholy K, Chen CY, **Lee KH**, Intini G. Outcome of early dental implant placement versus other dental implant placement protocols: A systematic review and meta-analysis. J Periodontol. 2019 05; 90(5):493-506. PMID: 30395355.
- Koch GK, Hamilton A, Wang K, Herschdorfer L, **Lee KH**, Gallucci GO, Friedland B. Dimensional accuracy of cone beam CT with varying angulation of the jaw to the X-ray beam. Dentomaxillofac Radiol. 2019 May; 48(4):20180319. PMID: 30407848.
- Liu SH, Bobb JF, **Lee KH**, Gennings C, Claus Henn B, Bellinger D, Austin C, Schnaas L, Tellez-Rojo MM, Hu H, Wright RO, Arora M, Coull BA. Lagged kernel machine regression for identifying time windows of susceptibility to exposures of complex mixtures. Biostatistics. 2018 07 01; 19(3):325-341. PMID: 28968676.

# Publication[中文]

# Information Reference

-  homepage: http://www.hsph.harvard.edu/kyu-ha-lee
-  googlescholar: https://scholar.google.com/citations?hl=zh-CN&user=6cxMccMAAAAJ
-  aminer: 

# Notes