---
bio-current:
  name-cn: 
  name-en: Matthew Stephens
  email: 
    - stephens@statistics.uchicago.edu # 如果有多个邮箱，请都填写上
  sex: male
  birth-date: # yyyy 到年即可
  university: University of Chicago 
  school: Dept, of Statistics [https://stat.uchicago.edu/] # 格式：学院名称[学院官网链接]
  major: 
  title-raw: Ralph W. Gerard Professor# 主页原始字符串
  title: Professor
  interests: # 分点罗列，依次以 ‘-’ 开头
    - Sparsity, shrinkage, and false discovery rates, particularly for complex inter-related datasets.
    - Factor Analysis, dimension reduction, and estimation of large covariance matrices.
    - Clustering methods, and generalizations (eg grade of membership)
    - Applications of multi-scale and wavelet methods to genomic data
    - Reproducible research and open science
  homepage: 
    - https://stat.uchicago.edu/people/profile/matthew-stephens/ # 如果有多个主页，请都填写上
    - http://stephenslab.uchicago.edu/
  github: 
  googlescholar: https://scholar.google.com/citations?hl=en&user=qOQFhUkAAAAJ
  aminer: https://www.aminer.org/profile/matthew-stephens/5440f3d9dabfae805a70edfa
  status: 在职
  last-update: 2021-09-25
edu-phd:  # 读博经历
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: # 格式：导师名 [邮箱/网址]
  degree: # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: 
  school: 
  date-start: 
  date-end: 
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: 
  research: 
  software: 
  project: 
  blog: 
  arxiv: 
  linkedin: 
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: 
  school: 
  major: 
  email: 
  homepage: # 机构内学者主页
  date-start: 
  title: 
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Matthew Stephens](https://d3qi0qp55mx5f5.cloudfront.net/stat/i/people/Faculty/Stephens_Matthew_600x600.jpg?mtime=1565184607)

# Biography[English]

# Biography[中文]

# Interests[English]

My lab works on a wide variety of problems at the interface of Statistics and Genetics. We often tackle problems where novel statistical methods are required, or can learn something new compared with existing approaches. Thus, much of our research involves developing new statistical methodology, many of which have a non-trivial computational component. And because data sets are getting larger and larger our work often involves modern methods for "high-dimensional statistics". Our work often makes extensive use of Bayesian hierarchical models to borrow information across data sets or sampling units.

Recently my lab has been increasingly focussed on making its research more open, reproducible and extensible. This is because I see this as the first step towards greater cooperation of scientists to achieve common goals.

See http://github.com/stephenslab/ash for an example of a recent project I conducted "in the open". And see https://jdblischak.github.io/workflowr/ for an R package we have developed to help students and others make research websites of their analyses. In learning to do research this way, my lab uses git for version control, github for sharing code, and knitr and RStudio for helping make our R analyses clear and share-able.

Current research interests include:

- Sparsity, shrinkage, and false discovery rates, particularly for complex inter-related datasets.
- Factor Analysis, dimension reduction, and estimation of large covariance matrices.
- Clustering methods, and generalizations (eg grade of membership)
- Applications of multi-scale and wavelet methods to genomic data
- Reproducible research and open science

# Interests[中文]

# Education[English]

# Education[中文]

# Awards[English]

# Awards[中文]

# Talks[English]

# Talks[中文]

# Work experience[English]

# Work experience[中文]

# Publication[English]

## 2021

- H Shim, Z Xing, E Pantaleo, F Luca, R Pique-Regi and M Stephens. [Multi-scale Poisson process approaches for differential expression analysis of high-throughput sequencing data.](https://arxiv.org/abs/2106.13634) arXiv:2106.13634. [R package](https://github.com/heejungshim/multiseq) | [source code implementing the analyses](https://github.com/heejungshim/multiseq_Shim_et_al)

- P Carbonetto, A Sarkar, Z Wang and M Stephens. [Non-negative matrix factorization algorithms greatly improve topic model fits.](https://arxiv.org/abs/2105.13440) arXiv:2105.13440. [R package](https://github.com/stephenslab/fastTopics) | [companion source code repository](https://github.com/stephenslab/fastTopics-experiments)

- A Sarkar and M Stephens. [Separating measurement and expression models clarifies confusion in single cell RNA-seq analysis.](https://doi.org/10.1038/s41588-021-00873-4) *Nature Genetics* 53: 770-777. [bioRxiv preprint](https://doi.org/10.1101/2020.04.07.030007) | [Python package](https://github.com/aksarkar/scmodes) | [companion website](https://aksarkar.github.io/singlecell-modes) | [companion source code repository](https://github.com/aksarkar/singlecell-modes)

- W Wang and M Stephens. [Empirical Bayes matrix factorization.](https://jmlr.org/papers/v22/20-589.html) *Journal of Machine Learning Research* 22(120): 1-40. [arXiv preprint](https://arxiv.org/abs/1802.06931) | [R package](https://github.com/stephenslab/flashr) | [code used to produce results in paper](https://github.com/NKweiwang/flash_workflow)

- Z Xing, P Carbonetto and M Stephens. [Flexible signal denoising via flexible empirical Bayes shrinkage.](https://jmlr.org/papers/v22/19-042.html) *Journal of Machine Learning Research* 22(93): 1-28. [arXiv preprint](https://arxiv.org/abs/1605.07787) | [R package](https://github.com/stephenslab/smashr) | [accompanying code and data](https://github.com/stephenslab/smash-paper)

- A E White, K K Dey, M Stephens and T D Price. [Dispersal syndromes drive the formation of biogeographical regions, illustrated by the case of Wallace’s Line.](https://doi.org/10.1111/geb.13250) *Global Ecology and Biogeography* 30: 685–696. [accompanying code and data](https://kkdey.github.io/wallace_analysis)

- M C Ward, N E Banovich, A Sarkar, M Stephens and Yoav Gilad. [Dynamic effects of genetic variation on gene expression revealed following hypoxic stress in cardiomyocytes.](https://doi.org/10.7554/eLife.57345) *eLife* 10: e57345.

## 2020

- Z Zhang, K Luo, Z Zou, M Qiu, J Tian, L Sieh, H Shi, Y Zou, G Wang, J Morrison, A C Zhu, M Qiao, Z Li, M Stephens, X He and C He. [Genetic analyses support the contribution of mRNA N6-methyladenosine (m6A) modification to human disease heritability.](https://doi.org/10.1038/s41588-020-0644-z) *Nature Genetics* 52: 939–949.

- S Kim-Hellmuth et al. [Cell type–specific genetic regulation of gene expression across human tissues.](https://doi.org/10.1126/science.aaz8528) *Science* 369(6509): eaaz8528.

- J Morrison, N Knoblauch, J Marcus, M Stephens and X He. [Mendelian randomization accounting for correlated and uncorrelated pleiotropic effects using genome-wide summary statistics.](https://doi.org/10.1038/s41588-020-0631-4) *Nature Genetics* 52: 740–747. [bioRxiv preprint](https://doi.org/10.1101/682237) | [code](https://jean997.github.io/cause)

- C J Hsiao, P Tung, J D Blischak, J Burnett, K Barr, K K Dey, M Stephens and Y Gilad. [Characterizing and inferring quantitative cell cycle phase in single-cell RNA-seq data analysis.](http://www.genome.org/cgi/doi/10.1101/gr.247759.118) *Genome Biology* 30: 611-621. [bioRxiv preprint](https://doi.org/10.1101/526848) | [R package](https://github.com/jhsiao999/peco)

- G Wang, A Sarkar, P Carbonetto and M Stephens. [A simple new approach to variable selection in regression, with application to genetic fine mapping.](https://doi.org/10.1101/501114) *Journal of the Royal Statistical Society, Series B* 82: 1273-1300. [R package](https://github.com/stephenslab/susieR) | [accompanying code and data resources](https://github.com/stephenslab/susie-paper)

- Y Kim, P Carbonetto, M Stephens and M Anitescu. [A fast algorithm for maximum likelihood estimation of mixture proportions using sequential quadratic programming.](https://doi.org/10.1080/10618600.2019.1689985) *Journal of Computational and Graphical Statistics* 29: 261-273. [arXiv preprint](https://arxiv.org/abs/1806.01412) | [accompanying code resources](https://github.com/stephenslab/mixsqp-paper) | [R package](https://github.com/stephenslab/mixsqp)

## 2019

- J D Blischak, P Carbonetto and M Stephens. [Creating and sharing reproducible research code the workflowr way.](https://doi.org/10.12688/f1000research.20843.1) *F1000Research* 8: 1749 [version 1; peer review: 3 approved]. [R package on CRAN](https://cran.r-project.org/package=workflowr) | [R package on GitHub](https://github.com/jdblischak/workflowr)

- M Lu and M Stephens. [Empirical Bayes estimation of normal means, accounting for uncertainty in estimated standard errors.](https://arxiv.org/abs/1901.10679) arXiv:1901.10679. [accompanying code and data](https://github.com/stephenslab/EBNM-moderated-t)

- D Gerard and M Stephens. [Unifying and generalizing methods for removing unwanted variation based on negative controls.](https://doi.org/10.5705/ss.202018.0345) *Statistica Sinica* forthcoming. [arXiv preprint](https://arxiv.org/abs/1705.08393) | [R package](https://github.com/dcgerard/vicar) | [code used to produce results in paper](https://github.com/dcgerard/ruvb_sims)

- M C Turchin and M Stephens. [Bayesian multivariate reanalysis of large genetic studies identifies many new associations.](https://doi.org/10.1371/journal.pgen.1008431) *PLoS Genetics* 15(10): e1008431. [bioRxiv preprint](https://doi.org/10.1101/638882) | [R package](https://github.com/mturchin20/bmass)

- S Zhao, J Liu, P Nanga, Y Liu, A E Cicek, N Knoblauch, C He, M Stephens and X He. [Detailed modeling of positive selection improves detection of cancer driver genes.](https://doi.org/10.1038/s41467-019-11284-9) *Nature Communications* 10: 3399. [bioRxiv preprint](https://doi.org/10.1101/366823) | [accompanying code and data resources](https://szhao06.bitbucket.io/driverMAPS-documentation/docs)

- A Sarkar, P-Y Tung, J D Blischak, J E Burnett, Y I Li, M Stephens and Y Gilad. [Discovery and characterization of variance QTLs in human induced pluripotent stem cells.](https://doi.org/10.1371/journal.pgen.1008045) *PLoS Genetics* 15(4): e1008045. [accompanying code and data](https://github.com/jdblischak/singlecell-qtl)

- H Al-Asadi, K K Dey, J Novembre and M Stephens. [Inference and visualization of DNA damage patterns using a grade of membership model.](https://doi.org/10.1093/bioinformatics/bty779) *Bioinformatics* 35(8): 1292-1298. [R package](https://github.com/kkdey/aRchaic)

- A E White, K K Dey, D Mohan, M Stephens and T D Price. [Regional influences on community structure across the tropical-temperate divide.](https://doi.org/10.1038/s41467-019-10253-6) *Nature Communications* 10: 2646. [R package](https://github.com/kkdey/ecostructure)

- H Al-Asadi, D Petkova, M Stephens and J Novembre. [Estimating recent migration and population size surfaces.](https://doi.org/10.1371/journal.pgen.1007908) *PLoS Genetics* 15(1): e1007908. [software](https://github.com/halasadi/MAPS)

- S M Urbut, G Wang, P Carbonetto and M Stephens. [Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions.](http://dx.doi.org/10.1038/s41588-018-0268-8) *Nature Genetics* 51(1): 187-195. [bioRxiv preprint](http://dx.doi.org/10.1101/096552) | [R package](https://github.com/stephenslab/mashr) | [accompanying code and data resources](https://github.com/stephenslab/gtexresults)

## 2018

- K K Dey and M Stephens. [CorShrink: Empirical Bayes shrinkage estimation of correlations, with applications.](https://doi.org/10.1101/368316) bioRxiv doi:10.1101/368316. [R package](https://github.com/kkdey/CorShrink) | [accompanying code and data resources](https://kkdey.github.io/CorShrink-pages/workflow_project.html)

- L Sun and M Stephens. [Solving the empirical Bayes normal means problem with correlated noise.](https://arxiv.org/abs/1812.07488) arXiv:1812.07488. [R package](https://github.com/LSun/cashr) | [accompanying code and data resources](https://github.com/LSun/cashr_paper)

- L F V Ferrão, R G Ferrão, M A G Ferrão, A Fonseca, P Carbonetto, M Stephens and A A F Garcia. [Accurate genomic prediction of Coffea canephora in multiple environments using whole-genome statistical models.](https://doi.org/10.1038/s41437-018-0105-y) *Heredity* 122, 261-275. [data](https://doi.org/10.5061/dryad.1139fm7)

- K K Dey, D Xie and M Stephens. [A new sequence logo plot to highlight enrichment and depletion.](https://doi.org/10.1186/s12859-018-2489-3) *BMC Bioinformatics* 19: 473. [R package](https://github.com/kkdey/Logolas) | [code used to produce results in paper](https://github.com/kkdey/Logolas-paper)

- X Zhu and M Stephens. [Large-scale genome-wide enrichment analyses identify new trait-associated genes and pathways across 31 human phenotypes.](https://doi.org/10.1038/s41467-018-06805-x) *Nature Communications* 9: 4361. [software](https://github.com/stephenslab/rss) | [online results](http://xiangzhu.github.io/rss-gsea)

- D Gerard and M Stephens. [Empirical Bayes shrinkage and false discovery rate estimation, allowing for unwanted variation.](http://dx.doi.org/10.1093/biostatistics/kxy029) *Biostatistics* 21(1), 15–32. [R package](https://github.com/dcgerard/vicar) | [code used to produce results in paper](https://github.com/dcgerard/mouthwash_sims)

- M C Ward, S Zhao, K Luo, B J Pavlovic, M M Karimi, M Stephens and Y Gilad. [Silencing of transposable elements may not be a major driver of regulatory evolution in primate iPSCs.](https://doi.org/10.7554/eLife.33084) *eLife* 7: e33084.

- D Gerard, L F V Ferrão, A A F Garcia and M Stephens. [Genotyping polyploids from messy sequencing data.](https://doi.org/10.1534/genetics.118.301468) *Genetics* 210(3), 789-807. [R package](https://cran.r-project.org/package=updog) | [code used to produce results in paper](https://github.com/dcgerard/reproduce_genotyping)

- J Smith, G Coop, M Stephens and J Novembre. [Estimating time to the common ancestor for a beneficial allele.](https://doi.org/10.1093/molbev/msy006) *Molecular Biology and Evolution* 35(4): 1003–1017.

## 2017

- P Carbonetto, X Zhou and, M Stephens. [varbvs: fast variable selection for large-scale regression.](https://arxiv.org/abs/1709.06597) arXiv:1709.06597. [software](https://github.com/pcarbo/varbvs)

- X Zhu and M Stephens. [Bayesian large-scale multiple regression with summary statistics from genome-wide association studies.](http://stephenslab.uchicago.edu/assets/papers/Zhu2017.pdf) [*Annals of Applied Statistics* 11(3): 1561-1592.](http://dx.doi.org/10.1214/17-AOAS1046) [Supplemental materials](http://stephenslab.uchicago.edu/assets/papers/Zhu2017-supplement.pdf) | [Software](http://github.com/stephenslab/rss) | [Preprocessed height data](https://doi.org/10.5281/zenodo.1443565)

- I E Schor, J F Degner, D Harnett, E Cannavo, F P Casale, H Shim, D A Garfield, E Birney, M Stephens, O Stegle, and E E M Furlong. [Promoter shape varies across populations and affects promoter evolution and expression noise.](http://stephenslab.uchicago.edu/assets/papers/Schor2017.pdf) [*Nature Genetics* 49(4): 550-558.](http://dx.doi.org/10.1038/ng.3791) [Supplementary Text and Figures](http://stephenslab.uchicago.edu/assets/papers/Schor2017-supplement.pdf)

- K Dey, C Hsiao, and M Stephens. [Visualizing the structure of RNA-seq expression data using grade of membership models.](http://dx.doi.org/10.1371/journal.pgen.1006599) *PLoS Genetics* 13(3): e1006599. [R package on Github](https://github.com/kkdey/CountClust) | [R package on Bioconductor](http://bioconductor.org/packages/CountClust/) | [code used to produce results in paper](https://stephenslab.github.io/count-clustering)

- M Stephens. [False discovery rates: a new deal.](https://doi.org/10.1093/biostatistics/kxw041) *Biostatistics* 18(2): 275-294. [Supplementary Materials](http://stephenslab.uchicago.edu/assets/papers/Stephens2017-supplement.pdf) | [R package on CRAN](https://cran.r-project.org/package=ashr) | [R package on Github](https://github.com/stephens999/ashr) | [code used to produce results in paper](https://github.com/stephenslab/ash)

## 2016

- M Lu and M Stephens. [Variance adaptive shrinkage (vash): Flexible empirical Bayes estimation of variances.](http://stephenslab.uchicago.edu/assets/papers/lu-stephens-2016.pdf) [*Bioinformatics* 32(22): 3428-3434.](http://dx.doi.org/10.1093/bioinformatics/btw483) [Supplementary Materials](http://stephenslab.uchicago.edu/assets/papers/lu-stephens-2016-supplement.pdf) | [Software](https://github.com/mengyin/vashr)

- A Raj, S Wang, H Shim, A Harpak, Y I Li, B Englemann, M Stephens, Y Gilad, and J K Pritchard. [Thousands of novel translated open reading frames in humans inferred by ribosome footprint profiling.](http://dx.doi.org/10.7554/eLife.13328) *eLife* 2016;5:e13328.

- D Petkova, J Novembre, and M Stephens. [Visualizing spatial population structure with estimated effective migration surfaces.](http://stephenslab.uchicago.edu/assets/papers/Petkova2016.pdf) [*Nature Genetics* 48: 94-100.](http://dx.doi.org/10.1038/ng.3464) [Supplementary Text and Figures](http://stephenslab.uchicago.edu/assets/papers/Petkova2016-supplement.pdf) | [Software](https://github.com/dipetkov/eems)

## 2015

- S Mondol, I Moltke, J Hart, M Keigwin, L Brown, M Stephens, and S K Wasser. [New evidence for hybrid zones of forest and savanna elephants in Central and West Africa.](http://stephenslab.uchicago.edu/assets/papers/Mondol2015.pdf) [*Mol Ecol* 24(24): 6134-6147, December 2015.](http://dx.doi.org/10.1111/mec.13472)

- Y Shiraishi, G Tremmel, S Miyano, and M Stephens. [A simple model-based approach to inferring and visualizing cancer mutation signatures.](http://stephenslab.uchicago.edu/assets/papers/Shiraishi2015.pdf) [*PLoS Genetics* 11(12): e1005657, December 2015.](http://dx.doi.org/10.1371/journal.pgen.1005657)

- A Raj, H Shim, Y Gilad, J K Pritchard, and M Stephens. [msCentipede: Modeling heterogeneity across genomic sites improves accuracy in the inference of transcription factor binding.](http://stephenslab.uchicago.edu/assets/papers/Raj2015.pdf) [*PLoS ONE* 10(9): e0138030.](http://dx.doi.org/10.1371/journal.pone.0138030) [Software](http://rajanil.github.io/msCentipede/)

- H Shim and M Stephens. [Wavelet-based genetic association analysis of functional phenotypes arising from high-throughput sequencing assays.](http://stephenslab.uchicago.edu/assets/papers/Shim2015.pdf) [*Annals of Applied Statistics* 9(2): 665-686.](http://dx.doi.org/10.1214/14-AOAS776) [Software](https://github.com/heejungshim/WaveQTL)

- K G Ardlie et al. [The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans.](http://www.sciencemag.org/content/348/6235/648.short) *Science* 348(6235): 648-660, May 2015.

- H Shim, D I Chasman, J D Smith, S Mora, P M Ridker, D A Nickerson, R M Krauss, and M Stephens. [A multivariate genome-wide association analysis of 10 LDL subfractions, and their response to statin treatment, in 1868 Caucasians.](http://dx.doi.org/10.1371/journal.pone.0120758) *PLoS ONE* 10(4): e0120758. [Software](https://github.com/heejungshim/mvBIMBAM) | [Data](http://stephenslab.uchicago.edu/assets/data/Shim-et-al-2015/index.html)

- Z Gao, D Waggoner, M Stephens, C Ober, and M Przeworski. [An estimate of the average number of recessive lethal mutations carried by humans.](http://dx.doi.org/10.1534/genetics.114.173351) *Genetics* 199(4): 1243-1254.

- J Tung, X Zhou, S C Alberts, M Stephens, and Y Gilad. [The genetic architecture of gene expression levels in wild baboons.](http://dx.doi.org/10.7554/eLife.04729#.dpuf) *eLife* 2015;4:e04729.

## 2014

- X Zhou, C E Cain, M Myrthil, N Lewellen, K Michelini, E R Davenport, M Stephens, J K Pritchard, and Y Gilad. [Epigenetic modifications are associated with inter-species gene expression variation in primates.](http://www.genomebiology.com/2014/15/12/547) *Genome Biology* 15(12): 547, December 2014.

- X Wen and M Stephens. [Bayesian methods for genetic association analysis with heterogeneous subgroups: From meta-analyses to gene-environment interactions.](http://stephenslab.uchicago.edu/assets/papers/WenStephens2014.pdf) [*Annals of Applied Statistics* 8(1): 176-203.](http://dx.doi.org/10.1214/13-AOAS695) [Supplementary Text](http://stephenslab.uchicago.edu/assets/papers/WenStephens2014S.pdf)

- X Zhou and M Stephens. [Efficient multivariate linear mixed model algorithms for genome-wide assocication studies.](http://www.xzlab.org/papers/2014_Zhou&Stephens_NM.pdf) *Nature Methods* 11: 407-409. [Software](http://www.xzlab.org/software.html)

- A Raj, M Stephens, and J K Pritchard. [fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets.](http://www.genetics.org/content/197/2/573.short) *Genetics* 197(2): 573-589, June 2014. [Software](https://rajanil.github.io/fastStructure)

## 2013

- L M Mangravite, B E Engelhardt, M W Medina, J D Smith, C D Brown, D I Chasman, B H Mecham, B Howie, H Shim, D Naidoo, Q Feng, M J Rieder, Y D Chen, J I Rotter, P M Ridker, J C Hopewell, S Parish, J Armitage, R Collins, R A Wilke, D A Nickerson, M Stephens, and R M Krauss. [A statin-dependent QTL for GATM expression is associated with statin-induced myopathy.](http://dx.doi.org/10.1038/nature12508) *Nature* 502: 377-380.

- P Carbonetto and M Stephens. [Integrated enrichment analysis of variants and pathways in genome-wide association studies indicates central role for IL-2 signaling genes in type 1 diabetes, and cytokine signaling genes in Crohn's disease.](http://dx.doi.org/10.1371/journal.pgen.1003770) *PLoS Genetics* 9(10): e1003770. [Software](http://github.com/pcarbo/bmapathway)

- F Luca, J C Maranville, A L Richards, D B Witonsky, M Stephens, and A Di Rienzo. [Genetic, functional and molecular features of glucocorticoid receptor binding.](http://dx.plos.org/10.1371/journal.pone.0061654) *PLoS ONE* 8(4): e61654.

- M Stephens. [A Unified framework for association analysis with multiple related phenotypes.](http://dx.doi.org/10.1371/journal.pone.0065245) *PLoS ONE* 8(7): e65245.

- T Flutre, X Wen, J Pritchard, and M Stephens. [A statistical framework for joint eQTL analysis in multiple tissues.](http://dx.doi.org/10.1371/journal.pgen.1003486) *PLoS Genetics* 9(5): e1003486. [Software](http://stephenslab.uchicago.edu/software.html#eqtlbma)

- X Zhou, P Carbonetto, and M Stephens. [Polygenic modeling with Bayesian sparse linear mixed models.](http://www.xzlab.org/papers/2013_Zhouetal_PG.pdf) *PLoS Genetics* 9(2): e1003264. [Software](http://www.xzlab.org/software.html)

## 2012

- Q Li, J K Eng, and M Stephens. [A likelihood-based scoring method for peptide identification using mass spectrometry.](http://dx.doi.org/10.1214/12-AOAS568) *Annals of Applied Statistics* 6(4): 1775-1794. [arXiv](http://arxiv.org/abs/1301.2467)

- A A Pai, C E Cain, O Mizrahi-Man, S De Leon, N Lewellen, J-B Veyrieras, J F Degner, D J Gaffney, J K Pickrell, M Stephens, J K Pritchard, and Y Gilad. [The Contribution of RNA Decay Quantitative Trait Loci to Inter-Individual Variation in Steady-State Gene Expression Levels.](http://dx.plos.org/10.1371/journal.pgen.1003000) *PLoS Genetics* 8(10), e1003000, October 2012.

- A B Hart, B E Engelhardt, M C Wardle, G Sokoloff, M Stephens, H de Wit, and A A Palmer. [Genome-Wide Association Study of d-Amphetamine Response in Healthy Volunteers Identifies Putative Associations, Including Cadherin 13 (CDH13).](http://dx.doi.org/10.1371/journal.pone.0042646) *PLoS ONE* 7(8): e42646.

- J Maranville, F Luca, M Stephens, and A Di Rienzo. [Mapping gene-environment interactions at regulatory polymorphisms: insights into mechanisms of phenotypic variation.](http://stephenslab.uchicago.edu/assets/papers/Maranville2012.pdf) *Transcription* 3(2): 56-62.

- B Howie, C Fuchsberger, M Stephens, J Marchini, and GR Abecasis. [Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.](http://stephenslab.uchicago.edu/assets/papers/Howie2012.pdf) *Nature Genetics* 44(8): 955-959.

- X Zhou and M Stephens. [Genome-wide efficient mixed model analysis for association studies.](http://www.xzlab.org/papers/2012_Zhou&Stephens_NG.pdf) *Nature Genetics* 44(7): 821-824. [Supplementary Text](http://home.uchicago.edu/xz7/publication/Zhou_and_Stephens_NG_2012_SI.pdf) | [Software](http://www.xzlab.org/software.html)

- A Q Fu, D P Genereux, R Stoger, A F Burden, C D Laird, and M Stephens. [Statistical inference of in vivo properties of human DNA methyltransferases from double-stranded methylation patterns.](http://dx.doi.org/10.1371/journal.pone.0032225) *PLoS ONE* 7(3): e32225. [Software](http://stephenslab.uchicago.edu/software.html#methylhmm)

- G H Perry, P Melsted, J C Marioni, Y Wang, R Bainer, J K Pickrell, K Michelini, S Zehr, A D Yoder, M Stephens, J K Pritchard, and Y Gilad. [Comparative RNA sequencing reveals substantial genetic variation in endangered primates.](http://dx.doi.org/10.1101/gr.130468.111) *Genome Biology* 22: 602-610, 2011.

- P Carbonetto and M Stephens. [Scalable variational inference for Bayesian variable selection in regression, and its accuracy in genetic association studies.](http://stephenslab.uchicago.edu/assets/papers/Carbonetto2012.pdf) [*Bayesian Analysis* 7(1): 73-108, March 2012.](https://projecteuclid.org/euclid.ba/1339616726) [Software](http://github.com/pcarbo/Variational-inference-for-Bayesian-variable-selection)

- J-B Veyrieras, D J Gaffney, J K Pickrell, Y Gilad, M Stephens, and J K Pritchard. [Exon-specific QTLs skew the inferred distribution of expression QTLs detected using gene expression array data.](http://dx.plos.org/10.1371/journal.pone.0030629) *PLoS One* 7(2), e30629, February 2012.

- J F Degner, A A Pai, R Pique-Regi, J-B Veyrieras, D J Gaffney, J K Pickrell, S De Leon, K Michelini, N Lewellen, G E Crawford, M Stephens, Y Gilad, and J K Pritchard. [DNase I sensitivity QTLs are a major determinant of human expression variation.](http://stephenslab.uchicago.edu/assets/papers/degner2012.pdf) *Nature* 482: 390-394, February 2012.

- D J Gaffney, J-B Veyrieras, J F Degner, R Pique-Regi, A A Pai, G E Crawford, M Stephens, Y Gilad, and J K Pritchard. [Dissecting the regulatory architecture of gene expression QTLs.](http://dx.doi.org/10.1186/gb-2012-13-1-r7) *Genome Biology* 13:R7, January 2012.

## 2011

- L E Mechanic, H-S Chen, C I Amos, N Chatterjee, N J Cox, R L Divi, R Fan, E L Harris, K Jacobs, P Kraft, S M Leal, K McAllister, J H Moore, D N Paltoo, M A Province, E M Ramos, M D Ritchie, K Roeder, D J Schaid, M Stephens, D C Thomas, C R Weinberg, J S Witte, S Zhang, S Zollner, E J Feuer, and E M Gillanders. [Next generation analytic tools for large scale genetic epidemiology studies of complex diseases.](http://dx.doi.org/10.1002/gepi.20652) *Genetic Epidemiology* 36(1): 22-35, December 2011.

- B Howie, J Marchini, and M Stephens. [Genotype imputation with thousands of genomes.](http://dx.doi.org/10.1534/g3.111.001198) *G3: Genes, Genomics, Genetics* 1(6): 457-470, November 2011.

- Y Guan and M Stephens. [Bayesian Variable Selection Regression for Genome-wide Association Studies, and other Large-Scale Problems.](http://www.e-publications.org/ims/submission/index.php/AOAS/user/submissionFile/6478?confirm=37332ec8) *Annals of Applied Statistics* 5(3): 1780-1815, September 2011.

- J C Maranville, F Luca, A L Richards, X Wen, D B Witonsky, S Baxter, M Stephens, and A Di Rienzo. [Interactions between Glucocorticoid Treatment and Cis-Regulatory Polymorphisms Contribute to Cellular Response Phenotypes.](http://dx.doi.org/10.1371/journal.pgen.1002162) *PLoS Genetics* 7(7): e1002162, July 2011.

- A Fledel-Alon, E M Leffler, Y Guan, M Stephens, G Coop, and M Przeworski. [Variation in Human Recombination Rates and Its Genetic Determinants.](http://dx.doi.org/10.1371/journal.pone.0020321) *PLoS ONE* 6(6): e20321, June 2011.

## 2010

- A Q Fu, D P Genereux, R Stoger, C D Laird, and M Stephens. [Statistical inference of transmission fidelity of DNA methylation patterns over somatic cell divisions in mammals.](http://stephenslab.uchicago.edu/assets/papers/Fu2010.pdf) *Annals of Applied Statistics* 4(2): 871-892, June 2010.

- Q Li, M MacCoss, and M Stephens. [A nested mixture model for protein identification using mass spectrometry.](http://stephenslab.uchicago.edu/assets/papers/Li2010.pdf) *Annals of Applied Statistics* 4(2): 962-987, June 2010.

- X Wen and M Stephens. [Using linear predictors to impute allele frequencies from summary or pooled genotype data.](http://stephenslab.uchicago.edu/assets/papers/Wen2010.pdf) *Annals of Applied Statistics* 4(3): 1158-1182, September 2010. [Software](http://stephenslab.uchicago.edu/software_pages/blimp/index.html)

- J Novembre and M Stephens. [Response to Cavalli-Sforza interview.](http://dx.doi.org/10.3378/027.082.0408) *Human Biology* 82(4): 469-470, August 2010.

- L B Barreiro, J C Marioni, R Blekham, M Stephens, and Y Gilad. [Functional comparison of innate immune signaling pathways in primates.](http://dx.doi.org/10.1371/journal.pgen.1001249) *PLoS Genetics* 6(12): e1001249, December 2010.

- B E Engelhardt and M Stephens. [Analysis of population structure: a unifying framework and novel methods based on sparse factor analysis.](http://dx.doi.org/doi:10.1371/journal.pgen.1001117) *PLoS Genetics* 6(9): e1001117. [Software](http://stephenslab.uchicago.edu/software.html)

- J K Pickrell, J C Marioni, A A Pai, J F Degner, B E Engelhardt, E Nkadori, J B Veyrieras, M Stephens, Y Gilad, and J K Pritchard. [Understanding mechanisms underlying human gene expression variation with RNA sequencing.](http://stephenslab.uchicago.edu/assets/papers/Pickrell2010.pdf) *Nature* 464(7289):768-72, Mar 2010.

- M J Barber, L M Mangravite, C L Hyde, D I Chasman, J D Smith, C A McCarty, X Li, R A Wilke, M J Rieder, P T Williams, P M Ridker, A Chatterjee, J I Rotter, D A Nickerson, M Stephens, and R M Krauss. [Genome-wide association of lipid-lowering response to statins in combined study populations.](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0009763) *PLoS ONE* Mar 2010. [Supplementary Results Data Page](http://stephenslab.uchicago.edu/BarberKrauss2010SuppData.html)

- R Blekhman, J C Marioni, P Zumbo, M Stephens, and Y Gilad. [Sex-specific and lineage-specific alternative splicing in primates.](http://stephenslab.uchicago.edu/assets/papers/Blekhman2010.pdf) *Genome Res* 20(2):180-9, Feb 2010.

## 2009

- S Basu, M Stephens, J S Pankow, and E A Thompson. [A Likelihood-Based Trait-Model-Free Approach for Linkage Detection of Binary Trait.](http://stephenslab.uchicago.edu/assets/papers/Basu2010.pdf) *Biometrics* 66(1):205-13, Mar 2009.

- M J Hubisz, D Falush, M Stephens, and J K Pritchard. [Inferring weak population structure with the assistance of sample group information.](http://stephenslab.uchicago.edu/assets/papers/Hubisz2009.pdf) *Molecular Ecology Resources* 9:1322-1332, 2009.

- M Stephens and D J Balding. [Bayesian statistical methods for genetic association studies.](http://stephenslab.uchicago.edu/assets/papers/Stephens2009.pdf) *Nat Rev Genet* 10(10):681-90, Oct 2009.

## 2008

- Y Guan and M Stephens. [Practical issues in imputation-based association mapping.](http://stephenslab.uchicago.edu/assets/papers/Guan2008.pdf) *PLoS Genet* 4(12), Dec 2008.

- J Novembre, T Johnson, K Bryc, Z Kutalik, A R Boyko, A Auton, A Indap, K S King, S Bergmann, M R Nelson, M Stephens, and C D Bustamante. [Genes mirror geography within Europe.](http://stephenslab.uchicago.edu/assets/papers/Novembre2008.pdf) *Nature* 456(7218):98-101, Nov 2008.

- J B Veyrieras, S Kudaravalli, S Y Kim, E T Dermitzakis, Y Gilad, M Stephens, and J K Pritchard. [High-resolution mapping of expression-QTLs yields insight into human gene regulation.](http://stephenslab.uchicago.edu/assets/papers/Veyrieras2008.pdf) *PLoS Genet* 4(10), Oct 2008.

- S K Wasser, W J Clark, O Drori, E S Kisamo, C Mailand, B Mutayoba, and M Stephens. [Combating the illegal trade in African elephat ivory with DNA forensics.](http://stephenslab.uchicago.edu/assets/papers/Wasser2008.pdf) *Conserv Biol* 22(4):1065-1071, Aug 2008.

- J C Marioni, C E Mason, S M Mane, M Stephens, and Y Gilad. [RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays.](http://stephenslab.uchicago.edu/assets/papers/Marioni2008.pdf) *Genome Res* 18(9):1509-1517, Sep 2008. [Software](http://www.ebi.ac.uk/~marioni/resources/GR_Analysis_120608.R)

- P Scheet and M Stephens. [Linkage disequilibrium-based quality control for large-scale genetic studies.](http://stephenslab.uchicago.edu/assets/papers/Scheet2008.pdf) *PLoS Genet* 4(8), 2008.

- A P Reiner, M J Barber, Y Guan, P M Ridker, L A Lange, D I Chasman, J D Walston, G M Cooper, N S Jenny, M J Rieder, J P Durda, J D Smith, J Novembre, R P Tracy, J I Rotter, M Stephens, D A Nickerson, and R M Krauss. [Polymorphisms of the HNF1A gene encoding hepatocyte nuclear factor-1 alpha are accosicated with c-reactive protein.](http://stephenslab.uchicago.edu/assets/papers/Reiner2008.pdf) *Am J Hum Genet* 82(5):1193-1201, May 2008.

- J Novembre and M Stephens. [Interpreting principal component analyses of spatial population genetic variation.](http://stephenslab.uchicago.edu/assets/papers/Novembre2008b.pdf) *Nat Genet* 40(5):646-649, May 2008. [Supplementary Materials](http://stephenslab.uchicago.edu/assets/papers/novembre.stephens.suppinfo.pdf)

## 2007

- The International HapMap Consortium. [A second generation human haplotype map of over 3.1 million SNPs.](http://stephenslab.uchicago.edu/assets/papers/HapmapIIb2007.pdf) *Nature* 449(7164):851-61, Oct 2007.

- B Servin and M Stephens. [Imputation-based analysis of association studies: candidate regions and quantitative traits.](http://stephenslab.uchicago.edu/assets/papers/Servin2007.pdf) *PLoS Genet* 3(7), Jul 2007. [Supplementary File S1](http://stephenslab.uchicago.edu/assets/papers/Servin2007S1.pdf) | [Supplementary File S2](http://stephenslab.uchicago.edu/assets/papers/Servin2007S2.pdf)

- D Falush, M Stephens, and J K Pritchard. [Inference of population structure using multilocus genotype data: dominant markers and null alleles.](http://stephenslab.uchicago.edu/assets/papers/Falush2007.pdf) *Molecular Ecology Notes* 7:574-578, Jul 2007.

- A Roychoudhury and M Stephens. [Fast and accurate estimation of the population-scaled mutation rate, theta, from microsatellite genotype data.](http://stephenslab.uchicago.edu/assets/papers/Roychoudhury2007.pdf) *Genetics* 176(2):1363-1366, Jun 2007.

- G Hellenthal and M Stephens. [msHOT: modifying Hudson's ms simulator to incorporate crossover and gene conversion hotspots.](http://stephenslab.uchicago.edu/assets/papers/Hellenthal2007.pdf) *Bioinformatics* 23(4):520-521, 2007.

- S K Wasser, C Mailand, R Booth, B Mutayoba, E Kisamo, B Clark, and M Stephens. [Using DNA to track the origin of the largest ivory seizure since the 1989 trade ban.](http://stephenslab.uchicago.edu/assets/papers/Wasser2007.pdf) *Proc Natl Acad Sci U S A* 104(10):4228-4233, Mar 2007.

## 2006

- T Du Raedt, M Stephens, I Heyns, H Brems, D Thijs, L Messiaen, K Stephens, C Lazaro, K Wimmer, H Kehrer-Sawatzki, D Vidaud, L Kluwe, P Marynen, and E Legius. [Conservation of hotspots for recombination in low-copy repeats associated with the NF1 microdeletion.](http://stephenslab.uchicago.edu/assets/papers/DuRaedt2006.pdf) *Nat Genet* 38(12):1419-1423, 2006.

- T R Bhangale, M Stephens, and D A Nickerson. [Automating resequencing-based detection of insertion-deletion polymorphisms.](http://stephenslab.uchicago.edu/assets/papers/Bhangale2006.pdf) *Nat Genet* 38(12):1457-1462, 2006.

- G Hellenthal and M Stephens. [Insights into recombination from population genetic variation.](http://stephenslab.uchicago.edu/assets/papers/Hellenthal2006a.pdf) *Curr Opin Genet Dev* 16(6):565-572, 2006.

- R Gottardo, J Besag, M Stephens, and A Murua. [Probabilistic segmentation and intensity estimation for microarray images.](http://stephenslab.uchicago.edu/assets/papers/Gottardo2006.pdf) *Biostatistics* 7(1):85-99, 2006.

- G Hellenthal, J K Pritchard, and M Stephens. [The effects of genotype-dependent recombination, and transmission asymmetry, on linkage disequilibrium.](http://stephenslab.uchicago.edu/assets/papers/Hellenthal2006b.pdf) *Genetics* 172(3):2001-2005, 2006.

- M Stephens, J S Sloan, P D Robertson, P Scheet, and D A Nickerson. [Automating sequence-based detection and genotyping of SNPs from diploid samples.](http://stephenslab.uchicago.edu/assets/papers/Stephens2006.pdf) *Nat Genet* 38(3):375-381, 2006.

- P Scheet and M Stephens. [A fast and flexible method for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.](http://stephenslab.uchicago.edu/assets/papers/Scheet2006.pdf) *Am J Hum Genet* 78(4):629-644, 2006.

- J Marchini, D Cutler, N Patterson, M Stephens, E Eskin, E Halperin, S Lin, Z S Qin, H M Munro, G R Abecasis, and P Donnelly. [A comparison of phasing algorithms for trios and unrelated individuals.](http://stephenslab.uchicago.edu/assets/papers/Marchini2006.pdf) *Am J Hum Genet* 78(3):437-450, 2006.

## 2005

- The International HapMap Consortium. [A haplotype map of the human genome.](http://stephenslab.uchicago.edu/assets/papers/HapMap2005.pdf) *Nature* 237(7063):1299-1320, 2005.

- M Stephens and P Scheet. [Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation.](http://stephenslab.uchicago.edu/assets/papers/Stephens2005.pdf) *Am J Hum Genet* 76(3):449-462, 2005.

## 2004

- D C Crawford, T Bhangale, N Li, G Hellenthal, M J Rieder, D A Nickerson, and M Stephens. [Evidence for substantial fine-scale variation in recombination rates across the human genome.](http://stephenslab.uchicago.edu/assets/papers/Crawford2004.pdf) *Nat Genet* 36(7):700-706, 2004.

- S E Ptak, A D Roeder, M Stephens, Y Gilad, S Paabo, and M Przeworski. [Absence of the TAP2 human recombination hotspot in chimpanzees.](http://stephenslab.uchicago.edu/assets/papers/Ptak2004.pdf) *PLoS Biology* 2(6):849-855, 2004.

- S K Wasser, A M Shedlock, K Comstock, E A Ostrander, B Mutayoba, and M Stephens. [Assigning African elephant DNA to geographic region of origin: applications to the ivory trade.](http://stephenslab.uchicago.edu/assets/papers/Wasser2004.pdf) *Proc Natl Acad Sci U S A* 41:14844-14852, 2004. [Software](https://github.com/stephens999/scat)

## 2003

- N Li and M Stephens. [Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.](http://stephenslab.uchicago.edu/assets/papers/Li2003.pdf) *Genetics* 165(4)2213-2233, 2003.

- M Stephens and P Donnelly. [A comparison of Bayesian methods for haplotype reconstruction from population genotype data.](http://stephenslab.uchicago.edu/assets/papers/Stephens2003a.pdf) *Am J Hum Genet* 73(5):1162-1169, 2003.

- D Falush, M Stephens, and J K Pritchard. [Inference of population structure from multilocus genotype data: linked loci and correlated allele frequencies.](http://stephenslab.uchicago.edu/assets/papers/Falush2003a.pdf) *Genetics* 164:1567-1587. 2003.

- D Falush, T Wirth, D Linz, J K Pritchard, M Stephens, M Kidd, M J Blaser, D Y Graham, S Vacher, G I Perez-Perez, Y Yamoka, F Megraud, K Otto, U Reichard, E Katzowitsch, X Wang, M Achtman, and S Suerbaum. [Traces of human migrations in Heliobacter pylori populations.](http://stephenslab.uchicago.edu/assets/papers/Falush2003b.pdf) *Science* 299(5612):1582-1585, 2003.

- M Stephens and P Donnelly. [Ancestral inference in population genetics models with selection.](http://stephenslab.uchicago.edu/assets/papers/Stephens2003b.pdf) *Australian and New Zealand Journal of Statistics* 45:901-931, 2003.

## 2001

- M Stephens, N J Smith, and P Donnelly. [A new statistical method for haplotype reconstruction from population data.](http://stephenslab.uchicago.edu/assets/papers/Stephens2001.pdf) *Am J Hum Genet* 68(4):978-989, 2001.

## 2000

- J K Pritchard, M Stephens, and P Donnelly. [Inference of population structure using multilocus genotype data.](http://stephenslab.uchicago.edu/assets/papers/Pritchard2000a.pdf) *Genetics* 155(2):945-959, 2000.

- J K Pritchard, M Stephens, N A Rosenberg, and P Donnelly. [Association mapping in structured populations.](http://stephenslab.uchicago.edu/assets/papers/Pritchard2000b.pdf) *Am J Hum Genet* 67:170-181, 2000.

- M Stephens and P Donnelly. [Inference in molecular population genetics.](http://stephenslab.uchicago.edu/assets/papers/Stephens2000a.pdf) *J R Stat Soc, Ser B* 62:605-655, 2000.

- M Stephens. [Dealing with label-switching in mixture models.](http://stephenslab.uchicago.edu/assets/papers/Stephens2000b.pdf) *J R Stat Soc, Ser B* 62:795-809, 2000.

- M Stephens. [Bayesian analysis of mixtures with an unknown number of components - an alternative to reversible jump methods.](http://stephenslab.uchicago.edu/assets/papers/Stephens2000c.pdf) *Annals of Statistics* 28(1), 2000.

- M Stephens. [Times on trees and the age of an allele.](http://stephenslab.uchicago.edu/assets/papers/Stephens2000d.pdf) *Theoretical Population Biology* 57:109-119, 2000.

## 1999

- M Stephens. [Problems with computational methods in population genetics.](http://stat.fi/isi99/proceedings/arkisto/varasto/step0444.pdf) *Bulletin of the International Statistical Institute* 52nd session, 1999.

# Publication[中文]

# Information Reference

- homepage: https://stat.uchicago.edu/people/profile/matthew-stephens/
- homepage: http://stephenslab.uchicago.edu/index.html
- googleschlor: https://scholar.google.com/citations?hl=en&user=qOQFhUkAAAAJ
- Aminer: https://www.aminer.org/profile/matthew-stephens/5440f3d9dabfae805a70edfa

# Notes