---
bio-current:
  name-cn: 
  name_en: Haiyan Huang
  email: 
    - 
    
              hhuang@stat.berkeley.edu
           # 如果有多个邮箱，请都填写上
  sex: female # male/female
  birth-date: # yyyy 到年即可
  university: University of California��Berkeley 
  school: Dept, of Statistics [https://statistics.berkeley.edu/people/faculty] # 格式：学院名称[学院官网链接]
  major: 
  title-raw: Professor# 主页原始字符串
  title: Professor # Associate Professor/Assistant Professor/Professor
  interests: 
    - high dimensional and integrative genomic data analysis
    - network modeling
    - hierarchical multi-label classification
    - translational bioinformatics
  # 分点罗列，依次以 ‘-’ 开头
  homepage: 
    - https://statistics.berkeley.edu/people/haiyan-huang 
    - https://www.stat.berkeley.edu/~hhuang/
    # 如果有多个主页，请都填写上
  github: 
  googlescholar: https://scholar.google.com/citations?user=iVavvW8AAAAJ&hl=en  
  aminer: https://www.aminer.cn/profile/haiyan-huang/53f5859edabfaeae64f8045b # 从这里查找 https://www.aminer.org/search/person
  status: 在职 # 选项如下：在读/在职/离职/退休/亡故
  last-update: # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: # 格式：导师名 [邮箱/网址]
  degree: # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: 
  school: 
  date-start: 
  date-end: 
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: 
  research:
  - https://hhuanglabweb.github.io/2019-02-16-dlspring19/
  - https://vcresearch.berkeley.edu/faculty/haiyan-huang
  software: 
  project: https://hhuanglabweb.github.io/project/
  blog: 
  arxiv: 
  linkedin: https://www.linkedin.com/in/haiyan-huang-04884446/?miniProfileUrn=urn%3Ali%3Afs_miniProfile%3AACoAAAm8HzgBwyZqTKKvffsYe0uhPMsvXveCPck
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: 
  school: 
  major: 
  email: 
  homepage: # 机构内学者主页
  date-start: 
  title: 
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Haiyan Huang](https://statistics.berkeley.edu/sites/default/files/styles/crop_person/public/faculty/hhuang.png?h=2608151f&itok=0cLElQHi)

# Biography[English]

# Biography[中文]

# Interests[English]

# Interests[中文]

# Education[English]

# Education[中文]

# Awards[English]

# Awards[中文]

# Talks[English]

# Talks[中文]

# Work experience[English]

# Work experience[中文]

# Publication[English]

# Publication[中文]

1.     Jiang T, Hall A, Eres M, Hemmatian Z, Qiao B, Zhou Y, Ruan Z, Couse AD, Heller WT, Huang H, de la Cruz MO, Rolandi M, Xu T (2020). Single-chain heteropolymers transport protons selectively and rapidly. Nature. 2020 Jan;577(7789):216-20.

2.     Liu K, Theusch E, Zhou Y, Ashuach T, Dose AC, Bickel PJ*, Medina MW*, Huang H* (2019). GeneFishing to reconstruct context specific portraits of biological processes. Proceedings of the National Academy of Sciences. 2019 Sep 17;116(38):18943-50.

3.     Chang M, Edmiston EK, Womer FY, Zhou Q, Wei S, Jiang X, Zhou Y, Ye Y, Huang H, Zuo XN, Xu K (2019). Spontaneous low-frequency fluctuations in the neural system for emotional perception in major psychiatric disorders: amplitude similarities and differences across frequency bands. Journal of psychiatry & neuroscience: JPN. 2019 Mar; 44(2):132.

4.     Hu ZT, Ye Y, Newbury PA, Huang H*, Chen B* (2019). AICM: A Genuine Framework for Correcting Inconsistency Between Large Pharmacogenomics Datasets. In PSB 2019 (pp. 248-259).

5.     Jiang H, Sohn L, Huang H*, Chen L* (2018). Single Cell Clustering Based on Cell-Pair Differentiability Correlation and Variance Analysis. Bioinformatics. 2018 May 16;1:11.

6.     Kang CC, Ward TM, Bockhorn J, Schiffman C, Huang H, Pegram MD, Herr AE (2018). Electrophoretic cytopathology resolves ERBB2 forms with single-cell resolution. NPJ precision oncology. 2018 Mar 22;2(1):10.

7.     Lin JM, Kang CC, Zhou Y, Huang H, Herr AE, Kumar S (2018). Linking invasive motility to protein expression in single tumor cells. Lab on a Chip. 2018;18(2):371-84.

8.     Pimentel H, Hu ZT, Huang H* (2018). Biclustering by Sparse Canonical Correlation Analysis. Quantitative Biology. 2018 Mar 1;6(1):56-67.

9.     Wang YXR, Liu K, Theusch E, Rotter JI, Medina MW, Waterman M*, Huang H* (2017). Generalized correlation measure using count statistics for gene expression data with ordered samples. Bioinformatics.  2017 Oct 12;34(4):617-24.

10.  Huang H*, Yu B* (2017). Data wisdom in computational genomics research. Statistics in Biosciences. 2017; 9(2):646-61.

11.  Chang M, Womer FY, Edmiston KE, Bai C, Zhou Q, Jiang X, Wei S, Wei Y, Ye Y, Huang H, He Y, Xu K, Tang Y, Wang F (2017). Neurobiological Commonalities and Distinctions Among Three Major Psychiatric Diagnostic Categories: A Structural MRI Study. Schizophrenia Bulletin. 2017 Jun 13.

12.  Shi F, Huang H* (2017). Identifying Cell Subpopulations and Their Genetic Drivers from Single-Cell RNA-Seq Data Using a Biclustering Approach. Journal of Computational Biology. 2017 Jul 1;24(7):663-74.                                             

13.  Schiffman C, Lin C, Shi F, Chen L, Sohn L, Huang H* (2017). SIDEseq: A Cell Similarity Measure Defined by Shared Identified Differentially Expressed Genes for Single-cell RNA Sequencing data. Statistics in Biosciences. June 2017, Volume 9, Issue 1, pp 200–216.     

14.  Sinkala E, Sollier-Christen E, Renier C, Rosàs-Canyelles E, Che J, Heirich K, Duncombe TA, Vlassakis J, Yamauchi KA, Huang H, Jeffrey SS, Herr AE (2017). Profiling protein expression in circulating tumour cells using microfluidic western blotting. Nature Communications, Mar 23;8:14622.

Wang YXR, Jiang K, Feldman LJ, Bickel PJ, Huang H* (2015). Inferring Gene-Gene Interactions and Functional Modules Using Sparse Canonical Correlation Analysis. Annals of Applied Statistics. 9(1): 300-323.
16.  National Research Council Committee on Predictive-Toxicology Approaches for Military Assessments of Acute Exposures (2015). “National Academy of Sciences: Application of Modern Toxicology Approaches for Predicting Acute Toxicity for Chemical Defense.” Washington, DC: The National Academies Press. (I served as an NRC Committee member)

Wang YXR, Waterman MS*, Huang H* (2014). Gene Coexpression Measures in Large Heterogeneous Samples Using Count Statistics. Proc Natl Acad Sci. USA. 111(46):16371-6 [Paper Link] Commentary [Link]
Kim K, Bolotin E, Theusch E, Huang H, Medina MW, Krauss RM (2014). Prediction of LDL Cholesterol Response to Statin using Transcriptomic and Genetic Variation. Genome Biology. 15(9): 460. [Paper Link]
Li JJ, Huang H*, Bickel PJ*, Brenner S* (2014). Comparison of D. Melanogaster and C. Elegans Developmental Stages, Tissues, and Cells by modENCODE RNA-seq Data.  Genome Research. 24: 1084-1101. [Paper Link]
Jiang CR, Liu CC, Zhou XJ, Huang H* (2014). Optimal Ranking in Multi-label Classification Using Local Precision Rates. Statistica Sinica. 24: 1547-1570. [preprint Link]
Gerstein M, et al. (2014). Comparative Analysis of the Transcriptome across Distant Species. Nature. 512: 445-448. doi:10.1038/nature13424. [Paper Link]
Boyle AP, et al. (2014). Comparative Analysis of Regulatory Information and Circuits across Distant Species. Nature. 512: 453-456. doi:10.1038/nature13668. [Paper Link]
Wang YXR, Huang H*. (2014) Review on Statistical Methods for Gene Network Reconstruction Using Expression Data. Journal of Theoretical Biology. 362:53-61. doi:10.1016/j.jtbi.2014.03.040 [Paper Link]
Xiong H, Brown JB, Boley N, Bickel PJ, Huang H* (2014). DE-FPCA: Testing Gene Differential Expression and Exon Usage through Functional Principal Component Analysis. In S. Datta and D. Nettleton, editors, "Statistical Analysis of Next  Generation Sequence Data (Frontiers in Probability and Statistical Science)." Springer. New York. (ISBN-13: 978-3319072111; ISBN-10: 3319072110)
Li JJ, Huang H, Qian M, Zhang X (2014). Transcriptome Analysis Using Next-Generation Sequencing. Chapter in “Advanced Medical Statistics”. 2nd Edition. To appear. (Publication Date: December 20, 2014 | ISBN-10: 9814583294 | ISBN-13: 978-9814583299) [Book Link]
Chapman MR, Balakrishnan KR, Li J, Conboy MJ, Huang H, Mohanty SK, Jabart E, Hack J, Conboy IM, Sohn LL (2013). Sorting single satellite cells from individual myofibers reveals heterogeneity in cell-surface markers and myogenic capacity. Integrative Biology. 5(4):692-702.
ENCODE Consortium Project (2012). An Integrated Encyclopedia of DNA Elements in the Human Genome. Nature. 489, 57-74.
Kim K, Teng S, Jiang K, Feldman L, Huang H* (2012). Using biologically interrelated experiments to identify pathway genes in arabidopsis. Bioinformatics. 28(6), 815-822. [Paper Link]
Gao Q, Ho C, Jia Y, Li JJ, Huang H* (2012). Biclustering of Linear Patterns in Gene Expression Data (CLiP). Journal of Computational Biology. 19(6), 619-631.
Chapman MR, Balakrishnan K, Conboy MJ, Mohanty SK, Jabart E, Li J, Huang H,  Hack J, Conboy IM, and Sohn LL (2012). Label-free screening of niche-to-niche variation in Satellite stem cells using functionalized pores. A chapter in Nanopores for Bioanalytical Applications: Proceedings of the International Conference, Eds. J. Edel and T. Albrecht, RSC Publishing, pp. 38-42.
Li JJ, Jiang CR, Brown BJ, Huang H*, Bickel PJ* (2011). Sparse Linear Modeling of RNA-seq Data for Isoform Discovery and Abundance Estimation. Proc Natl Acad Sci. USA. 108 (50) 19867-19872. [Paper Link]
Li Q, Brown JB, Huang H, Bickel PJ. (2011). Measuring Reproducibility of High-throughput Experiments. Annals of Applied Statistics. 5(3), 1752-1779. [Paper Link]
Li Y, Huang H and Cai L (2011) Prediction of Transcriptional Regulatory Networks for Retinal Development. A chapter in book “Computational Biology and Applied Bioinformatics” edited by Lopes HS and Cruz LM.
Durinck S, Ho C, Wang NJ, Liao W, Jakkula LR, Collisson EA, Pons J, Chan SW, Lam ET, Chu C, Park K, Hong S, Hur JS, Huh N, Neuhaus IM, Yu SS, Grekin RC, Mauro  TM, Cleaver JE, Kwok P, LeBoit PE, Getz G, Cibulskis K, Aster JC, Huang H, Purdom  E, Li J, Bolund L, Arron ST, Gray JW, Spellman PT, Cho RJ (2011). Temporal Dissection of Tumorigenesis in Primary Cancers. Cancer Discovery, 1:137-143.
Xu N, Bickel PJ, Huang H* (2010). Genome-wide Detection of Transcribed Regions through Multiple RNA Tiling Array Analysis. International Journal of Systems and Synthetic Biology. 1(2) 155-170.
Huang H*, Liu C, Zhou XJ* (2010). Bayesian Approach to Transforming Public Gene Expression Repositories into Disease Diagnosis Databases. Proc Natl Acad Sci. USA. 107 (15) 6823-6828. [Paper Link]
·     This paper is selected for issue highlight by PNAS: http://www.pnas.org/content/107/15/6553.full.pdf+html (it is under the title "Gene databases mined for diagnoses").

·     This paper has been selected for Faculty of 1000 Biology (http://www.f1000biology.com) and evaluated by Dr. Russ Altman from Stanford University: http://www.f1000biology.com/article/id/3925957.  (Faculty of 1000 Biology is an award-winning online service that highlights and evaluates the most interesting papers published in the biological sciences, based on the recommendations of over 2000 of the world's top researchers.)

·     This paper has also been reported in the following news reports:

·       GenomeWeb daily news. “Team Develops Proof-of-Principle Diagnostic Database for Applying Public Gene Expression Data”. March 22, 2010.

·       National Cancer Institute. Research News. “Mathematical Modeling Turns Gene Expression Data into Disease Diagnostics” http://physics.cancer.gov/news/2010/april/po_news_b.asp

·       Biocentury and Nature publishing group. “GEO: world of diagnostic potential,” Haas, M.J. SciBX 3(14); April 8, 2010

Bickel PJ, Boley N, Brown JB, Huang H, Zhang NR (2010). Subsampling Methods for Genomic Inference. Annals of Applied Statistics. 4(4) 1660-1697.
(authors ordered alphabetically)

Jiang K, Zhu T, Diao Z, Huang H, Feldman LJ. (2010). The Maize Root Stem Cell Niche: A Partnership between Two Sister Cell Populations. Planta, 231(2):411-24. [Paper Link]
Bickel P, Brown B, Huang H, Li Q (2009). An overview of recent developments in genomics and associated statistical methods. Philosophical Transactions of the Royal Society A 367, 4313-4337. [Paper Link]
(authors ordered alphabetically)

Teng S, Huang H* (2009). A statistical framework to infer functional gene associations from multiple biologically interrelated microarray experiments. Journal of the American Statistical Association, June 2009, Vol. 104, No. 486. [Paper Link]
Wang F, Jiang T, Sun Z, Teng SL, Luo X, Zhu Z, Zang Y, Zhang H, Yue W, Hong N, Huang H, Blumberg H, Zhang, D (2009). Neuregulin 1 genetic variation and anterior cingulum integrity in patients with schizophrenia and healthy controls. Journal of Psychiatry & Neuroscience, 2009 May;34(3):181-6. [Paper Link]
Liu C, Hu J, Kalakrishnan M, Huang H*, Zhou XJ* (2009) Integrative disease classification based on cross-platform microarray Data. BMC Bioinformatics, 2009 Jan;10 Suppl 1:S25. [Paper Link]
Carbonaro A, Mohanty SK, Huang H, Godley LA and Sohn LL (2008) Cell characterization using a protein-functionalized Pore. Lab Chip, 8(9):1478-85. [Paper Link]
Huang H, Cai L, Wong WH. (2008) Clustering analysis of SAGE transcription profiles using a Poisson approach. Methods Mol Biol. 2008; 387:185-98. (Book Chapter) [Paper Link]
Huang Y, Li H, Hu H, Yan X, Waterman MS, Huang H, Zhou XJ (2007). Systematic discovery of functional modules and context-specific functional annotation of human genome. Bioinformatics, 23(13):i222-i229. [Paper Link]
ENCODE Consortium (2007)._ Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature. 447, 799-816. [Paper Link]
Kim K, Zhang S, Jiang K, Cai L, Lee IB, Feldman LJ, Huang H* (2007). Measuring similarities between gene expression profiles through data transformations. BMC Bioinformatics, 8:29 (highly accessed paper). [Paper Link]
Jiang K, Zhang S, Lee S, Tsai G, Kim K, Huang H, Zhu T, Feldman LJ (2006). Transcription profile analyses identify genes and pathways central to root cap functions in Maize. Plant Molecular Biology, 60(3):343-63. [Paper Link]
Huang H, Kim K (2006). Unsupervised clustering analysis of gene expression, Chance, vol. 19, No.3. [Paper Link]
Zhou XJ, Kao MJ, Huang H, Wong A, Nunez-Iglesias J, Aparicio OM, Morgan TE, Wong WH (2005). Functional annotation and network reconstruction through cross-platform integration of microarray data. Nature Biotechnology, 23(2):238-43. [Paper Link]
Zhao X, Huang H, Speed T (2005). Finding short DNA motifs using permuted Markov models. Journal of Computational Biology, 12(6): 894-906 (journal version of the 2004 RECOMB paper; numbered as 16 below) [Paper Link]
Huang H, Kao MJ, Zhou X, Liu JS, Wong WH (2004). Determination of local statistical significance of patterns in Markov sequences with application to promoter element identification.” Journal of Computational Biology, 11(1):1-14. [Paper Link]
Cai L#, Huang H#, Blackshaw S, Liu JS, Cepko CL, Wong WH (2004). Clustering analysis of SAGE data using a Poisson approach. Genome Biology, 5(7):R51. [Paper Link]
#Joint first authors

Zhao X, Huang H, Speed T (2004). Finding short DNA motifs using permuted Markov models. Proceedings of RECOMB 2004. [Paper Link]
Blackshaw S, Harpavat S, Trimarchi J, Cai L, Huang H, Kuo W, Fraioli R, Cho S, Yung R, Asch E, Wong WH, Cepko CL (2004). Genomic analysis of mouse retinal development. PLoS Biol, 2(9):E247. [Paper Link]
Allinen M, Beroukhim R, Cai L, Brennan C, Domenici CJ, Huang H, Porter D, Hu M, Chin L, Richardson A, Schnitt S, Sellers W, Polyak K (2004). Molecular characterization of the tumor microenvironment in breast cancer. Cancer Cell, 6(1):17-32. [Paper Link]
Lippert RA, Huang H, Waterman MS (2002). Distributional regimes for the number of k-word matches between two random sequences. Proc Natl Acad Sci. USA, 99(22):13980-9. [Paper Link]
Huang H (2002). Error bounds on multivariate normal approximations for word count statistics. Advances in Applied Probability, 34(3): 559-586. [Paper Link]
 

 
Books Edited
 

“Research in Computational Molecular Biology” (11th Annual International Conference, RECOMB 2007), edited by Terry Speed and Haiyan Huang, Published by Springer. [Book Link]
“Inaugural DahShu Data Science Symposium: Computational Precision Health (CPH 2017)”, edited by Haiyan Huang, Hua Tang, et. al.. Special Issue published by the Journal of Computational Biology.
 

 

 

Articles Submitted or Manuscripts in Preparation
 

Chang M, Edmiston KE, Womer FY, Zhou Q, Wei S, Jiang X, Ye Y, Huang H$$, Xu K, Tang Y, Wang F (2017).  Spontaneous Low-Frequency Fluctuations in Neural System for Emotional Perception in Major Psychiatric Diagnostic Categories: Amplitude Similarities and Differences across Frequency Bands. Submitted.
Liu K, et. al. (2017) Bagged semi-supervised gene clustering for context-specific gene pathway analysis. 
Ho C, Jiang CR, Lee W, Huang H* (2017). Optimal decision making in hierarchical multi-label classification.
Ren H, Qi C, Ma Q, Sun X, Wei Q, Liang JW, Li G, Zhang Z, Ren J, Yan J, Li G,   Xu W, Li X, Bai C, She X, Wu G, Huang H and McCormick S (2017). Fixation of expression divergences by natural selection in Arabidopsis coding genes.

# Information Reference

https://www.stat.berkeley.edu/~hhuang/
https://hhuanglabweb.github.io/2019-02-16-dlspring19/
https://vcresearch.berkeley.edu/faculty/haiyan-huang
https://hhuanglabweb.github.io/project/

# Notes