---
bio-current:
  name-cn: 
  name_en: Elizabeth Purdom
  email: 
    - 
    
              epurdom@stat.berkeley.edu
           # 如果有多个邮箱，请都填写上
  sex: female # male/female
  birth-date: # yyyy 到年即可
  university: University of California��Berkeley 
  school: Dept, of Statistics [https://statistics.berkeley.edu/people/faculty] # 格式：学院名称[学院官网链接]
  major: 
  title-raw: Associate Professor # 主页原始字符串
  title:  Associate Professor # Associate Professor/Assistant Professor/Professor
  interests: 
    - computational biology
    - bioinformatics
    - statistics
    - data analysis
    - sequencing
    - cancer genomics
  # 分点罗列，依次以 ‘-’ 开头
  homepage: 
    - https://statistics.berkeley.edu/people/elizabeth-purdom 
    - http://www.stat.berkeley.edu/~epurdom
    # 如果有多个主页，请都填写上
  github: 
  googlescholar: https://scholar.google.com/citations?hl=en&user=WGgu4SAAAAAJ&view_op=list_works%A0 
  aminer: https://www.aminer.cn/profile/elizabeth-purdom/53f46d11dabfaedd74e8470e # 从这里查找 https://www.aminer.org/search/person
  status: 在职 # 选项如下：在读/在职/离职/退休/亡故
  last-update: # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: # 格式：导师名 [邮箱/网址]
  degree: # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: 
  school: 
  date-start: 
  date-end: 
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: 
  research: https://vcresearch.berkeley.edu/faculty/elizabeth-purdom
  software: 
  project: 
  blog: 
  arxiv: 
  linkedin: https://www.linkedin.com/in/elizabeth-purdom-78a13463/?miniProfileUrn=urn%3Ali%3Afs_miniProfile%3AACoAAA1eDloBASXn0YIIFGjtzQRaH1QqB64FumY
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: 
  school: 
  major: 
  email: 
  homepage: # 机构内学者主页
  date-start: 
  title: 
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Elizabeth Purdom](https://statistics.berkeley.edu/sites/default/files/styles/crop_person/public/faculty/dsc_0122_small.jpg?h=86163206&itok=GrutTR-u)

# Biography[English]

# Biography[中文]

# Interests[English]

My interests are in statistical problems arising in the field of molecular biology and genetics and in particular high-throughput genomic experiments. I am currently working on methodologies for analyzing gene expression and alternative splicing using microarrays and ultra high-throughput sequencing. Methodologically, I am interested in high-dimensional and multivariate techniques that are relevant in this context.

# Interests[中文]

# Education[English]

# Education[中文]

# Awards[English]

# Awards[中文]

# Talks[English]

# Talks[中文]

# Work experience[English]

# Work experience[中文]

# Publication[English]

Bioinformatic and Statistical Methodology

Gong B, Zhou Y, and Purdom E (under review) "Cobolt: Joint analysis of multimodal single-cell sequencing data" link
Roux de Bezieux H, Street K, Fischer S, Van den Berge K, Chance R, Risso D, Gillis J, Ngai J, Purdom E, and Dudoit S (under review) "Improving replicability in single-cell RNA-Seq cell type discovery with Dune" link
Hicks SC, Liu R, Ni Y, Purdom E, Risso D (2021) "mbkmeans: fast clustering for single cell data using mini-batch k-means" PLOS Computational Biology 17(1): e1008625 link
Varoquaux N., Purdom E. (2020) "A pipeline to analyse time-course gene expression data" F1000Research 9:1447 link
Gong B., Purdom E. (2019) MethCP: Differentially Methylated Region Detection with Change Point Models. In: Cowen L. (eds) Research in Computational Molecular Biology. RECOMB 2019. Lecture Notes in Computer Science, vol 11467. link
El Karoui N and Purdom E (2019) "The bootstrap, covariance matrices and PCA in moderate and high-dimensions" Proceedings of Machine Learning Research: The 22nd International Conference on Artificial Intelligence and Statistics. 89:2115-2124. link
Cole MB, Risso D, Wagner A, DeTomaso D, Ngai J, Purdom E, Dudoit S, Yosef N (2019) "Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq" Cell Systems 8(4): 315-328. link
Risso D, Purvis L, Fletcher R, Das D, Ngai J, Dudoit S, Purdom E (2018) "clusterExperiment and RSEC: A Bioconductor package and framework for clustering of single-cell and other large gene expression datasets" PLoS Comput Biol. 2018 Sep 4;14(9):e1006378 link
Street K, Risso D, Fletcher RB, Das D, Ngai J, Yosef N, Purdom E, and Dudoit S (2018) "Slingshot: Cell lineage and pseudotime inference for single-cell transcriptomics" BMC Genomics 19(1): 477. link
El Karoui N, and Purdom E (2018) "Can we trust the bootstrap in high-dimension?" Journal of Machine Learning Research 19 (2018) 1-66 link
Perraudeau F, Risso D, Street K, Purdom E, and Dudoit S (2017) "Bioconductor workflow for single-cell RNA sequencing: Normalization, dimensionality reduction, clustering, and lineage inference" F1000Research 6:1158. link
Johnson M and Purdom E (2017) "Clustering of mRNA-Seq data based on alternative splicing patterns," Biostatistics (2017) 18 (2): 295-307. link
Ruddy S, Johnson M, Purdom E (2016) "Shrinkage of dispersion parameters in the binomial family, with application to differential exon skipping," Annals of Applied Statistics 10: 690-725 link
Purdom E, Ho C, Grasso CS, Quist MJ, Cho RJ, and Spellman P (2013) "Methods and challenges in timing chromosomal abnormalities within cancer samples," Bioinformatics. 29(24):3113-20. link
Purdom E (2012) "Annotation-Free Estimates of Gene-Expression from mRNA-Seq," Technical Report 825, Department of Statistics, UC Berkeley link
Purdom E (2011) "Analysis Of A Data Matrix And A Graph: Metagenomic Data And The Phylogenetic Tree", Ann. Appl. Stat. 5(4): 2326-2358. link
Wang XV, Verhaak RGW, Purdom E, Spellman PT, Speed TP (2011). "Unifying gene expression measures from multiple platforms using factor analysis" PloS one, 6(3) link
Bullard J, Purdom E, Hansen K, Dudoit S. (2010) "Evaluation of Statistical Methods for Normalization and Differential Expression in mRNA-Seq Experiments", BMC Bioinformatics 11:94. link
Purdom E. (2008) "Analyzing Data with Graphs: Metagenomic Data and the Phylogenetic Tree" Technical Report 766, Department of Statistics, UC Berkeley. link
Purdom E, Simpson K, Robinson M, Conboy J, Lapuk A. and Speed T. (2008) "FIRMA: A Method for Detection of Alternative Splicing from Exon Array Data" Bioinformatics 24(15): 1707-1714. link Data
Purdom E and Holmes S. (2005) "Error Distribution for Gene Expression Data", Statistical Applications in Genetics and Molecular Biology 4(1): Article 16. http://www.bepress.com/sagmb/vol4/iss1/art16
Back to top

Applied and Collaborative

BRAIN Initiative Cell Census Network (BICCN) (under review) "A multimodal cell census and atlas of the mammalian primary motor cortex" link
Yao Z, Liu H, Xie F, Fischer S, et al (under review) "An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types" link
Brann DH, Tsukahara T, Weinreb C, Lipovsek M, Van den Berge K, Gong B, Chance R, Macaulay IC, Chou H-J, Fletcher RB, Das D, Street K, Roux de Bezieux H, Choi YG, Risso D, Dudoit S, Purdom E, Mill J, Hachem RA, Matsunami H, Logan DW, Goldstein BJ, Grubb MS, Ngai J, Datta SR (2020) "Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" Science Advances 6: eabc5801 link
Gao C, Montoya L, Xu L, Madera M, Hollingsworth J, Purdom E, Singan V, Vogel J, Hutmacher RB, Dahlberg JA, Coleman-Derr D, Lemaux PG, and Taylor JW (2020) "Fungal community assembly in drought-stressed sorghum shows stochasticity, selection, and universal ecological dynamics" Nature Communications 11:34. link
Varoquaux N, Cole B, Gao C, Pierroz G, Baker CR, Patel D, Madera M, Jeffers T, Hollingsworth J, Sievert J, Yoshinaga Y, Owiti JA, Singan VR, DeGraaf S, Xu L, Blow MJ, Harrison MJ, Visel A, Jansson C, Niyogi KK, Hutmacher R, Coleman-Derr D, O'Malley RC, Taylor JW, Dahlberg J, Vogel JP, Lemaux PG, and Purdom E (2019) "Transcriptomic analysis of field-droughted sorghum from seedling to maturity reveals biotic and metabolic responses" PNAS, 116(52): 27124-27132. link
Harirchian P, Lee J, Hilz S, Sedgewick AJ, Perez White BE, Kesling MJ, Mully T, Golovato J, Gray M, Mauro TM, Purdom E, Kim EA, Sbitany H, Bhutani T, Vaske CJ, Benz SC, Cho RJ, and Cheng JB (2019) "A20 and ABIN1 Suppression of a Keratinocyte Inflammatory Program with a Shared Single-Cell Expression Signature in Diverse Human Rashes," Journal of Investigative Dermatology, 139(6), 1264-1273. link
Gao C, Montoya L, Xu L, Madera M, Hollingsworth J, Purdom E, Hutmacher RB, Dahlberg JA, Coleman-Derr D, Lemaux PG, and Taylor JW (2018) "Strong succession in arbuscular mycorrhizal fungal communities" The ISME Journal link
Cheng JB, Sedgewick AJ, Finnegan AI, Harirchian P, Lee J, Kwon S, Fassett MS, Golovato J, Gray M, Ghadially R, Liao W, White BEP, Mauro TM, Mully T, Kim EA, Sbitany H, Neuhaus IM, Grekin RC, Yu SS, Gray JW, Purdom E, Paus R, Vaske CJ, Benz SC, Song JS, Cho RJ (2018). "Transcriptional Programming of Normal and Inflamed Human Epidermis at Single-Cell Resolution" Cell Reports, 25(4), 871-883. link
Cho RJ, Alexandrov LB, den Breems NY, Atanasova VS, Farshchian M, Purdom E, Nguyen TN, Coarfa C, Rajapakshe K, Prisco M, Sahu J, Tassone P, Greenawalt EJ, Collisson EA, Wu W, Yao H, Su X, Guttmann-Gruber C, Hofbauer PN, Hashmi R, Fuentes I, Benz SC, Golovato J, Ehli EA, Davis CM, Davies GE, Covington KR, Murrell DF, Salas-Alanis JC, Palisson F, Bruckner AL, Robinson W, Has C, Bruckner-Tuderman L, Titeux M, Jonkman MF, Rashidghamat E, Lwin SM, Mellerio JE, McGrath JA, Bauer JW, Hovnanian A, Tsai KY, South AP (2018) "APOBEC mutation drives early-onset squamous cell carcinomas in recessive dystrophic epidermolysis bullosa" Science Translational Medicine 10 (455): eaas9668 link
Gadye L, Das D, Sanchez MA, Street K, Baudhuin A, Wagner A, Cole MB, Choi YG, Yosef N, Purdom E, Dudoit S, Risso D, Ngai J, and Fletcher RB (2017) "Injury Activates Transient Olfactory Stem Cell States with Diverse Lineage Capacities" Cell Stem Cell 21(6): 775-790 link
Naylor D, DeGraaf S, Purdom E, and Coleman-Derr D (2017) "Drought and host selection influence bacterial community dynamics in the grass root microbiome" ISME Journal 11: 2691-2704 link
Fletcher RB, Das D, Gadye L, Street KN, Baudhuin A, Wagner A, Cole MB, Flores Q, Choi YG, Yosef N, Purdom E, Dudoit S, Risso D and Ngai J (2017) "Deconstructing Olfactory Stem Cell Trajectories at Single Cell Resolution" Cell Stem Cell 20(6):817-830. link
Sanborn Z, Chung J, Purdom E, Wang NJ, Kakavand H, Wilmott J, Butler T, Thompson JF, Mann GJ, Haydu LE, Saw RP, Busam KJ, Lo R, Collisson EA, Hur JS, Spellman P, Cleaver JE, Gray JW, Huh N, Murali R, Scolyer RA, Bastian BC, Cho RJ (2015) "Phylogenetic analyses of melanoma reveal complex patterns of metastatic dissemination" PNAS 112: 10995-11000 link
Zheng CL, Wang NJ, Chung J, Moslehi H, Sanborn JZ, Hur JS, Collisson EA, Vemula SS, Naujokas A, Chiotti KE, Cheng JB, Fassihi H, Blumberg AJ, Bailey CV, Fudem GM, Mihm FG, Cunningham BB, Neuhaus IM, Liao W, Oh DH, Cleaver JE, LeBoit PE, Costello JF, Lehmann AR, Gray JW, Spellman PT, Arron ST, Huh N, Purdom E, Cho RJ (2014) "Transcription Restores DNA Repair to Heterochromatin, Determining Regional Mutation Rates in Cancer Genomes" Cell Reports 9(4): 1228-1234. link
Purdom E, Restall C, Busuttil RA, Schluter H, Boussioutas A, Thompson EW, Anderson RL, Speed TP, Haviv I. (2013) "Determining epithelial contribution to in vivo mesenchymal tumour expression signature using species-specific microarray profiling analysis of xenografts." Genetics Research. 95(1):14-29.link
Xenograft specific cdfs: Human cdf, Mouse cdf
Couvillion M, Bounova G, Purdom E, Speed T, and Collins K. (2013) "A Tetrahymena Piwi bound to mature tRNA 3' fragments activates the exonuclease Xrn2 for RNA processing in the nucleus" Molecular Cell. 48:509-20. link
Heiser LM, Sadanandam A, Kuo WL, Benz SC, Goldstein TC, Ng S, Gibb WJ, Wang NJ, Ziyad S, Tong F, Bayani N, Hu Z, Billig JI, Dueregger A, Lewis S, Jakkula L, Korkola JE, Durinck S, Pepin F, Guan Y, Purdom E, Neuvial P, Bengtsson H, Wood KW, Smith PG, Vassilev LT, Hennessy BT, Greshock J, Bachman KE, Hardwicke MA, Park JW, Marton LJ, Wolf DM, Collisson EA, Neve RM, Mills GB, Speed TP, Feiler HS, Wooster RF, Haussler D, Stuart JM, Gray JW, and Spellman PT (2012). "Subtype and pathway specific responses to anticancer compounds in breast cancer,"" Proc Natl Acad Sci U S A. 109(8):2724-9. link
The Cancer Genome Atlas Research Network (2011) "Integrated genomic analyses of ovarian carcinoma," Nature. 474(7353):609-15. link
Durinck S, Ho C, Wang NJ, Liao W, Jakkula LR, Collisson EA, Pons J, Chan SW, Lam ET, Chu C, Park K, Hong SW, Hur JS, Huh N, Neuhaus IM, Yu SS, Grekin RC, Mauro TM, Cleaver JE, Kwok PY, LeBoit PE, Getz G, Cibulskis K, Aster JC, Huang H, Purdom E, Li J, Bolund L, Arron ST, Gray JW, Spellman PT, and Cho RJ (2011). "Temporal Dissection of Tumorigenesis in Primary Cancers." Cancer Discovery. 1:137-143. link
Lapuk A, Marr H, Jakkula L, Pedro H, Bhattacharya S, Purdom E, Hu Z, Simpson K, Pachter L, Durinck S, Wang N, Parvin B, Fontenay G, Speed T, Garbe J, Stampfer M, Bayandorian H, Dorton S, Clark TA, Schweitzer A, Wyrobek A, Feiler H, Spellman P, Conboy J, Gray JW. (2010) "Exon-level microarray analyses identify alternative splicing programs in breast cancer", Molecular Cancer Research 8(7):961-74 link
Verhaak R, Hoadley K, Purdom E, Wang V, Qi Y, Wilkerson M, Miller CR , Ding L, Golub T, Mesirov JP, Alexe G, Lawrence M, O'Kelly M, Tamayo P, Weir BA, Gabriel S, Winckler W, Gupta S, Jakkula L, Feiler HS, Hodgson JG, James CD, Sarkaria JN, Brennan C, Kahn,A, Spellman PT, Wilson RK, Speed TP, Gray JW, Meyerson M, Getz G, Perou C, Hayes DN and The Cancer Genome Atlas Research Network (2010) "Integrated Genomic Analysis Identifies Clinically Relevant Subtypes of Glioblastoma Characterized by Abnormalities in PDGFRA, IDH1, EGFR, and NF1", Cancer Cell 17(1): 98-110. link
The Cancer Genome Atlas Research Network (2008) "Comprehensive Genomic Characterization Defines Human Glioblastoma Genes and Core Pathways" Nature 455(7216): 1061-1068. link
Bik E, Eckburg P, Gill S, Nelson K, Purdom E, Francois F, Perez-Perez G, Blaser M, and Relman D. (2006) "Molecular Analysis of the Bacterial Microbiota in the Human Stomach", PNAS 103(1): 732-737. link
Eckburg P, Bik E, Bernstein C, Purdom E, Dethlefsen L, Sargent M, Gill S, Nelson K, and Relman D. (2005) "Diversity of the Human Intestinal Microbial Flora", Science 308: 1635-1638. link
Xu T, Xhu C.-T, Purdom E, Dang D, Ilsley D, Guo Y, Weber J, Holmes S.P. and Lee P.P (2004) "Microarray Analysis Reveals Differences in Gene Expression of Circulating CD8+ T Cells in Melanoma Patients and Healthy Donors", Cancer Research 64: 3661-3667. link

# Publication[中文]

# Information Reference

https://www.stat.berkeley.edu/~epurdom/publications.html
https://statistics.berkeley.edu/people/elizabeth-purdom 
http://www.stat.berkeley.edu/~epurdom

# Notes