---
bio-current:
  name_en: Jeffrey Thorne
  email: 
    - thorne@ncsu.edu # 如果有多个邮箱，请都填写上
  sex: male # male/female
  birth-date: # yyyy 到年即可
  university: North Carolina State University 
  school: Department of Statistics [https://statistics.sciences.ncsu.edu/]# 格式：学院名称[学院官网链接]
  major: 
  title-raw: Professor # 主页原始字符串
  title: Professor # Associate Professor/Assistant Professor/Professor
  interests: # 分点罗列，依次以 ‘-’ 开头
    - Bioinformatics
    - Genetics
  homepage: 
    - https://statistics.sciences.ncsu.edu/people/thorne/ # 如果有多个主页，请都填写上
    - http://statgen.ncsu.edu/thorne/thorne.html
  github: 
  googlescholar:  
  aminer: https://www.aminer.cn/profile/jeffrey-l-thorne/53f45661dabfaeee22a3411e # 从这里查找 https://www.aminer.org/search/person
  status: 在职 # 选项如下：在读/在职/离职/退休/亡故
  last-update: # yyyy-mm-dd 最近一次信息更新日期
edu-phd:  # 读博经历
  university: University of Washington
  school: Genetics
  email: 
  date-start: 
  date-end: 1991
  advisor: # 格式：导师名 [邮箱/网址]
  degree: phd # phd
edu-master: # 硕士经历，没有或找不到，可不填
  university: 
  school: 
  date-start: 
  date-end: 
  advisor:
edu-bachelor:  # 本科经历，没有或找不到，可不填
  university: 
  school: 
  major: 
  date-start: 
  date-end: 
page-other:   # 其他有用的链接，部分可从学者主页子栏目获得
  publication: https://brcwebportal.cos.ncsu.edu/thorne/mypublications.html
  research: 
  software: https://brcwebportal.cos.ncsu.edu/thorne/multidivtime.html
  project: 
  blog: 
  arxiv: 
  linkedin: 
  weibo:
  twitter:
  wikipedia:
  baidu-baike:
collaboration: # 合作研究，关注学者和其他哪些学科的人合作，具体研究哪些主题
  - 
    with: # 合作者
    project: # 研究主题
  - 
    with: 
    project: 
group: # 所属团队，学者可能有不同的兴趣小组，可以列上去
job-faculty-1: # 所属机构，若有多个增加编号即可，字段填写参看示例文件
  university: North Carolina State University 
  school: Bioinformatics Research Center
  major: 
  email: 
  homepage: # 机构内学者主页
  date-start: 
  title: 
  type: 
job-post-doc: # 博士后研究员，字段填写参看示例文件，若无可不填写
  university: 
  school: 
  email: 
  date-start: 
  date-end: 
  advisor: 
---

# Profile

![Jeffrey Thorne](https://statistics.sciences.ncsu.edu/wp-content/uploads/sites/21/2019/06/jeff_thorne.jpg)

# Biography[English]

Dr. Thorne regularly teaches Bioinformatics II (ST590C), a core course in the Bioinformatics graduate curriculum. His research focus is evolutionary inference from interspecific data and the reconciliation of such inference techniques with population genetics; developing statistical tools for studying molecular evolution using DNA and protein sequences; protein structure and protein evolution; and using molecular and fossil data to estimate the times since different species had common ancestors.

# Interests[English]

- Bioinformatics

- Genetics

# Education[English]

- **Ph.D** Genetics University of Washington 1991

# Awards[English]



# Talks[English]



# Work experience[English]



# Publication[English]

Xiang, J., Griffing, A., and Thorne, J.L. (2016)  A phylogenetic approach finds abundant interlocus gene conversion in yeast.  Molecular Biology and Evolution 33:2469--2476.



Lee, H.-J., Kishino, H., Rodrigue, N., and Thorne, J.L. (2016)  Grouping substitution types into different relaxed molecular clocks.  Phil. Trans. of Roy. Soc. B. 371: 20150141.

Lee, H.-J., Rodrigue, N., and Thorne, J.L. (2015)  Relaxing the Molecular Clock to Different Degrees for Different Substitution Types.  Molecular Biology and Evolution 32(8) 1948-1961.



Wang, K., Yu, S., Ji, X., Lakner, C., Griffing, A., and Thorne, J.L. (2015)  Roles of solvent accessibility and gene expression in modeling protein sequence evolution.  Evolutionary Bioinformatics 11:85-96.



Lassiter, E.S., Russ, C., Nusbaum, C., Zeng, Q., Saville, A., Olarte, R., Carbone, I., Hu, C.-H., Seguin-Orlando, A., Samaniego, J.A., Thorne, J.L., and Ristaino, J.B. (2015)  Mitochondrial genome sequences reveal evolutionary relationships of the *Phytophthora* 1c clade species.  Current Genetics DOI: 10.1007/s00294-015-0480-3.

Hobolth, A., and Thorne, J.L. (2014)  Sampling and summary statistics of endpoint-conditioned paths in DNA sequence evolution.  Chapter 12 (pages 247-276) in "Bayesian Phylogenetics: Methods, Algorithms, and Applications", edited by M.-H. Chen, L. Kuo, and P.O. Lewis. Chapman and Hall / CRC Mathematical and Computational Biology.

Liberles, D.A., Teichmann, S.A., Bahar, I., Bastolla, U., Bloom, J., Bornberg-Bauer, E., Colwell, L.J., de Koning, A.P.J., Dokholyan, N.V., Echave, J., Elofsson, A., Gerloff, D.L., Goldstein, R.A., Grahnen, J.A., Holder, M.T., Lakner, C., Lartillot, N., Lovell, S.C., Naylor, G., Perica, T., Pollock, D.D., Pupko, T., Regan, L., Roger, A., Rubinstein, N., Shakhnovich, E., Sjolander, K., Sunyaev, S., Teufel, A.I., Thorne, J.L., Thornton, J.W., Weinreich, D.M. and Whelan, S. (2012)  The interface of protein structure, protein biophysics, and molecular evolution.  Protein Science 21(6):769-785.



Thorne, J.L., Lartillot, L., Rodrigue, N., and Choi, S.C. (2012)  Codon models as a vehicle for reconciling population genetics with inter-specific sequence data.  Chapter 7 (pages 97-1100 In ``Codon Evolution''), edited by G.M. Cannarozzi and A. Schneider. Oxford University Press.

Yokoyama, K.D., Thorne, J.L., and Wray, G.A. (2011)  Coordinated genome-wide modifications within proximal promoter cis-regulatory elements during vertebrate evolution.  Genome Biology and Evolution 3:66-74.



Cartwright, R.A., Lartillot, L., and Thorne, J.L. (2011)  History can matter: Non-Markovian behavior of ancestral lineages.  Systematic Biology 60:276-290.

Choi, S.C., Stone, E.A., Kishino, H., and Thorne, J.L. (2009) Estimates of natural selection due to protein tertiary structure inform the ancestry of biallelic loci. Gene 441:45-52.

Choi, S.C., Redelings, B.D., and Thorne, J.L. (2008) Basing population genetic inferences and models of molecular evolution upon desired stationary distributions of DNA or protein sequences. Phil. Trans. R. Soc. B. 363:3931-3939.

Xiang, Q.-Y., Thorne, J.L., Seo, T.-K., Zhang, W., Thomas, D.T., and Ricklefs, R.E. (2008) Rates of nucleotide substitution in Cornaceae (Cornales) - pattern of variation and underlying causal factors. Mol. Phylo. Evol. 49:327-342.

Kitazoe, Y., Kishino, H., Hasegawa, M., Nakajima, N., Thorne, J.L., and Tanaka, M. (2008) Adaptive Threonine Increase in Transmembrane Regions of Mitochondrial Proteins in Higher Primates. PLoS ONE 3(10): e3343.

Seo, T.-K., Kishino, H., and Thorne, J.L. (2008) Bayesian divergence time estimation using codon model. Proceedings of the Institute of Statistical Mathematics. 56(1): 37-54. [(PDF file, In Japanese)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/seokishinothorne2008.pdf)

Thorne, J.L. (2008) Models and their evolution. Wissenschaftskolleg zu Berlin 2006/2007 Yearbook. [(PDF file of proofs)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thorne-wiko-proofs.pdf)

in, Y.-M., Fang, S.-C., Thorne, J.L. (2007) A TABU search algorithm for maximum parsimony phylogeny inference. Eur. J. Operational Research 176:1908-1917. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/LinFangThorne.pdf)

Thorne J.L., Goldman N. (2007) Probabilistic models for the study of protein evolution. Balding, Bishop, and Cannings, eds. Handbook of Statistical Genetics, Third Edition. Wiley and Sons. Volume I : 439-459.

Thorne, J.L., Choi, S. C., Yu, J., Higgs, P.G., Kishino, H. (2007) Population genetics without intraspecific data. Mol. Biol. Evol. 24:1667-1677. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/ThorneEtAlMBE2007.pdf)

Choi, S. C., Hobolth, A., Robinson, D. M., Kishino, H., Thorne, J.L. (2007) Quantifying the impact of protein tertiary structure on molecular evolution. Mol. Biol. Evol. 24:1769-1782. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/ChoiEtAlMBE2007.pdf)

Thorne, J.L. (2007) Protein evolution constraints and model-based techniques to study them. Current Opinion in Structural Biology 17:337-341. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornecurrop2007.pdf)

Yu, J., Thorne, J.L. (2006) Testing for spatial clustering of amino acid replacements within protein tertiary structure. J. Mol. Evol. 62(6):682-692. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/yuthorneJME2006.pdf)

Yu, J., Thorne J.L. (2006) Dependence among sites in RNA evolution. Mol. Biol. Evol. 23(8): 1525-1537. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/yuthorneMBE2006.pdf)

Thorne, J.L. and Kishino, H. (2005) Estimation of divergence times from molecular sequence data. In Statistical methods in molecular evolution. R. Nielsen, ed. Springer Verlag, pp.235-256 [(PDF format preprint)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/clockreview1215wfigs.pdf)

Seo, T.-K., Kishino, H., Thorne, J.L. (2005) Incorporating gene-specific variation when inferring and evaluating optimal evolutionary tree topologies from multilocus sequence data. Proc. Natl. Acad. Sci. U.S.A. 102(12): 4436-4441. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/seopnas05.pdf)

Thorne, J.L. (2004) Discussant (pages 584-585) on ``A Bayesian approach to DNA sequence segmentation'', a paper by Richard J. Boys and Daniel A. Henderson. Biometrics 60:573-588.  [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/boyshenderson04.pdf)

Sanderson, M.J., Thorne, J.L., Wikstrom, N., and Bremer, K. (2004) Molecular evidence on plant divergence times. Am. J.Bot., 91:1656-1665. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/SandersonAJB05.pdf)

Seo, T.-K., Kishino, H., and Thorne, J.L. (2004) Estimating absolute rates of synonymous and nonsynonymous nucleotide substitution in order to characterize natural selection and date species divergences. Mol. Biol. Evol. 21: 1201-1213. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/SeoMBE04.pdf)

choll EH, Thorne JL, McCarter JP, Bird DM. (2003) Horizontally transferred genes in plant-parasitic nematodes: A high-throughput genomic approach. Genome Biology 4(6):R39 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/scholl2003.pdf)

Robinson DM, Jones DT, Kishino H, Goldman N, Thorne JL (2003) Protein evolution with dependence among codons due to tertiary structure. Mol.Biol. Evol. 20(10):1692-1704 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/RobinsonMBE2003.pdf)

Hasegawa M, Thorne JL, Kishino H (2003) Time scale of eutherian evolution estimated without assuming constant rate of molecular evolution. Genes and Genetic Systems 78(4): 267-283. (review) [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/Hasegawa03GGS.pdf)

Wiegmann BM, Yeates DK, Thorne JL, Kishino H. (2003) Time Flies: A new molecular time-scale for fly evolution without a clock. Systematic Biology 52(6):745-756. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/wiegsysbio03.pdf)

Thorne, J.L. and Goldman, N. (2003) Probabilistic models for the study of protein evolution. (revised for second edition) In Handbook of Statistical Genetics, Second Edition. D.J. Balding, M. Bishop, and C. Cannings, eds.John Wiley and Sons, Inc.

Seo T-K, Thorne JL, Hasegawa M, Kishino H (2002) A Viral Sampling Design for Testing the Molecular Clock and for Estimating Evolutionary Rates and Divergence Times. Bioinformatics 18(1):115-123. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/seobioinf.pdf)

Thorne JL, Kishino H (2002) Divergence time estimation and rate evolution with multilocus data sets. Systematic Biology 51:689-702. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornekishino02.pdf)

Seo T-K, Thorne JL, Hasegawa M, Kishino H (2002) Estimation of effective population size of HIV-1 within a host: A pseudomaximum-likelihood approach. Genetics 160: 1283-1293.[(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/seogenetics.pdf)

Bird, D. McK., Clifton, S. W., Kepler, T., Kieber, J.J., Thorne, J. and C. H. Opperman (2002) Genomic dissection of a nematode-plant interaction:A tool to study plant biology. Plant Physiology 129:394-395.

Kishino H, Thorne JL (2002) Bayesian hierarchical model of rate of molecular evolution. Proceedings of the Institute of Statistical Mathematics.50(1): 17-31 (in Japanese).

Thorne JL, Goldman N (2001) Probabilistic Models for the Study of Protein Evolution. Balding, Bishop, and Cannings, eds. Handbook of Statistical Genetics. Wiley and Sons.

Kishino H, Thorne JL, Bruno WJ (2001) Performance of a divergence time estimation method under a probabilistic model of rate evolution. Mol Bio Evol, 18:352-361. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/kishino2001mbe.pdf)

Thorne JL (2000) Models of Protein Sequence Evolution and their Applications. Curr Opin Genet Dev, 10:602-605. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornecurrop.pdf)

Goldman N, Thorne JL, Jones DT (1998) Assessing the impact of secondary structure and solvent accessibility on protein evolution. Genetics 149:445-458.[(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/goldmangen98.pdf)

Lio P, Goldman N, Thorne JL, Jones DT (1998) PASSML: combining Protein Secondary Structure Prediction and Evolutionary Inference. Bioinformatics 14:726-733 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/liobioinf98.pdf)

Thorne JL, Kishino H, Painter IS (1998) Estimating the rate of evolution of the rate of molecular evolution. Mol Bio Evol 15:1647-1657. [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornekishino98.pdf)

Swofford D, Thorne JL, Felsenstein J, Wiegmann B (1996) The topology-dependent permutation test for monophyly does not test for monophyly. Sys Bio 45:575-579

Goldman N, Thorne JL, Jones DT (1996) Using evolutionary trees in protein secondary structure prediction and other comparative sequence analyses. J Mol Bio 263(2): 196-208 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/goldmanjmb96.pdf)

Thorne JL, Goldman N, Jones DT (1996) Combining protein evolution and secondary structure. Mol Bio Evol 13:666-673 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/tgj96mbe.pdf)

Thorne JL, Churchill GA (1995) Estimation and reliability of molecular sequence alignments. Biometrics 51:100-113 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornechurchill95.pdf)

Thorne JL, Kishino H, Felsenstein J (1992) Inching toward reality: An improved likelihood model of sequence evolution. J Mol Evol 34:3-16 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/tkfJME1992.pdf)

Thorne JL, Kishino H (1992) Freeing phylogenies from artifacts of alignment. Mol Bio Evol 9: 1148-1162 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/thornekishino92mbe.pdf)

Laird CD, Lamb MM, Sved J, Thorne J (1991) Modeling the inheritance and expression of the fragile-X syndrome, with emphasis on the X-inactivation imprinting model, pp. 228-251. In ``The Fragile-X Syndrome: Diagnosis, Treatment, and Research,'' R. Hagerman and A. Cronister, eds. John Hopkins University Press. (Review)

Thorne JL, Kishino H, Felsenstein J (1991) An evolutionary model for the maximum likelihood alignment of DNA sequences. J Mol Evol 33:114-124 [(PDF file)](ftp://statgen.ncsu.edu/pub/thorne/mypapers/tkfJME1991.pdf)

Laird CD, Lamb MM, Thorne JL (1990) Two progenitor cells for human oogonia inferred from pedigree data and the X-inactivation imprinting model of the fragile-X syndrome. Am J Hum Genet 46:696-719

Laird CD, Sved J, Thorne J, Lamb MM (1990) The X-inactivation imprinting model of the fragile-X syndrome: Annotated References, 1989. Chromosomes Today, 10:163-165 (Review)

# Information Reference

- peple: [North Carolina State University Bioinformatics Research Center (ncsu.edu)](https://brc.ncsu.edu/)
- homepage: 

# Notes